<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Advanced Statistical Analysis | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="In the molecular laboratory, statistical analysis is the mathematical framework that validates the reliability of test results. While the bench scientist performs the physical assay, statistics...">
<meta name="generator" content="bookdown 0.45 with bs4_book()">
<meta property="og:title" content="Advanced Statistical Analysis | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="In the molecular laboratory, statistical analysis is the mathematical framework that validates the reliability of test results. While the bench scientist performs the physical assay, statistics...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Advanced Statistical Analysis | MB">
<meta name="twitter:description" content="In the molecular laboratory, statistical analysis is the mathematical framework that validates the reliability of test results. While the bench scientist performs the physical assay, statistics...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-detection.html">Separation &amp; Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="" href="array-technology-1.html">Array Technology</a></li>
<li><a class="" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
<li class="book-part">Laboratory Operations</li>
<li><a class="" href="contamination.html">Contamination</a></li>
<li><a class="" href="prevention-1.html">Prevention</a></li>
<li><a class="" href="monitoring-detection-1.html">Monitoring &amp; Detection</a></li>
<li><a class="" href="elimination-1.html">Elimination</a></li>
<li><a class="" href="quality-assurance.html">Quality Assurance</a></li>
<li><a class="" href="specimen-handling.html">Specimen Handling</a></li>
<li><a class="" href="specimen-evaluation.html">Specimen Evaluation</a></li>
<li><a class="" href="nucleic-acid-evaluation.html">Nucleic Acid Evaluation</a></li>
<li><a class="" href="reagents.html">Reagents</a></li>
<li><a class="" href="assays.html">Assays</a></li>
<li><a class="" href="assay-troubleshooting-1.html">Assay Troubleshooting</a></li>
<li><a class="" href="result-calculation-interpretation-reporting-1.html">Result Calculation, Interpretation, &amp; Reporting</a></li>
<li><a class="" href="qc-pt.html">QC &amp; PT</a></li>
<li><a class="" href="assay-controls.html">Assay Controls</a></li>
<li><a class="" href="proficiency-testing.html">Proficiency Testing</a></li>
<li><a class="" href="equipment-instrumentation-1.html">Equipment &amp; Instrumentation</a></li>
<li><a class="" href="guidelines-regulations.html">Guidelines &amp; Regulations</a></li>
<li><a class="" href="test-system-categories-1.html">Test System Categories</a></li>
<li><a class="" href="regulations-standards-1.html">Regulations &amp; Standards</a></li>
<li><a class="" href="personnel.html">Personnel</a></li>
<li><a class="" href="continuing-education-1.html">Continuing Education</a></li>
<li><a class="" href="competency.html">Competency</a></li>
<li><a class="" href="safety.html">Safety</a></li>
<li><a class="" href="hazardous-materials.html">Hazardous Materials</a></li>
<li><a class="" href="biological.html">Biological</a></li>
<li><a class="" href="chemical.html">Chemical</a></li>
<li><a class="" href="laboratory-administration.html">Laboratory Administration</a></li>
<li><a class="" href="financial-management-1.html">Financial Management</a></li>
<li><a class="" href="budgets-1.html">Budgets</a></li>
<li><a class="" href="capital-equipment.html">Capital Equipment</a></li>
<li><a class="" href="cost-analysis-reimbursement-1.html">Cost Analysis &amp; Reimbursement</a></li>
<li><a class="" href="purchasing-inventory-1.html">Purchasing &amp; Inventory</a></li>
<li><a class="" href="operations-management-1.html">Operations Management</a></li>
<li><a class="" href="laboratory-information-system.html">Laboratory Information System</a></li>
<li><a class="" href="facilities-management-1.html">Facilities Management</a></li>
<li><a class="" href="intrainterdepartmental-relations-1.html">Intra/Interdepartmental Relations</a></li>
<li><a class="" href="personnel-management-1.html">Personnel Management</a></li>
<li><a class="" href="motivation-1.html">Motivation</a></li>
<li><a class="" href="staffing-productivity-1.html">Staffing &amp; Productivity</a></li>
<li><a class="" href="counseling-disciplinary-action-1.html">Counseling &amp; Disciplinary Action</a></li>
<li><a class="" href="quality-management-1.html">Quality Management</a></li>
<li><a class="active" href="advanced-statistical-analysis-1.html">Advanced Statistical Analysis</a></li>
<li><a class="" href="assay-method-instrument-selection.html">Assay, Method, &amp; Instrument Selection</a></li>
<li><a class="" href="assay-validation-verification.html">Assay Validation &amp; Verification</a></li>
<li><a class="" href="quantitative-calculations-1.html">Quantitative Calculations</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="advanced-statistical-analysis-1" class="section level1 unnumbered">
<h1>Advanced Statistical Analysis<a class="anchor" aria-label="anchor" href="#advanced-statistical-analysis-1"><i class="fas fa-link"></i></a>
</h1>
<p>In the molecular laboratory, statistical analysis is the mathematical framework that validates the reliability of test results. While the bench scientist performs the physical assay, statistics determine if the assay is valid, if the instrument is drifting, and if the patient result is clinically actionable. Advanced statistical analysis in this context moves beyond simple averages to encompass Method Validation, Diagnostic Accuracy, and Process Capability (Six Sigma)</p>
<div id="descriptive-statistics-precision-monitoring" class="section level2 unnumbered">
<h2>Descriptive Statistics &amp; Precision Monitoring<a class="anchor" aria-label="anchor" href="#descriptive-statistics-precision-monitoring"><i class="fas fa-link"></i></a>
</h2>
<p>Before complex analysis can occur, the laboratory must establish the baseline performance of its assays. This relies on descriptive statistics to quantify “Precision” - the ability of the assay to produce the same result repeatedly under unchanged conditions</p>
<ul>
<li>
<strong>Standard Deviation (SD)</strong>
<ul>
<li>SD measures the dispersion of data points around the Mean (average). In Molecular Quality Control (QC), the SD is the “yardstick” for acceptability</li>
<li>
<strong>Calculation:</strong> It quantifies how much the QC values typically vary. A small SD indicates high precision (tight clustering); a large SD indicates poor precision (scattered results)</li>
</ul>
</li>
<li>
<strong>Coefficient of Variation (%CV)</strong>
<ul>
<li>The %CV is the normalized measure of dispersion, allowing for the comparison of variability between different concentration ranges</li>
<li>
<strong>Formula:</strong> <span class="math inline">\(\%CV = (\text{Standard Deviation} / \text{Mean}) \times 100\)</span>
</li>
<li>
<strong>Molecular Application:</strong> This is critical in Quantitative PCR (Viral Loads). The standard deviation of a High Positive Control (1,000,000 copies/mL) will naturally be larger numerically than a Low Positive (1,000 copies/mL), but their %CVs should be similar. If the %CV at the Low Limit of Quantitation (LLoQ) exceeds a set threshold (e.g., 20%), the assay is no longer reliable at that level</li>
</ul>
</li>
</ul>
</div>
<div id="diagnostic-accuracy-clinical-performance" class="section level2 unnumbered">
<h2>Diagnostic Accuracy (Clinical Performance)<a class="anchor" aria-label="anchor" href="#diagnostic-accuracy-clinical-performance"><i class="fas fa-link"></i></a>
</h2>
<p>When validating a new test, the laboratory must determine how well it distinguishes between health and disease. This involves comparing the new test to a “Gold Standard” method to calculate contingency statistics. These metrics define the <strong>Clinical Sensitivity</strong> and <strong>Clinical Specificity</strong></p>
<ul>
<li>
<strong>Sensitivity (True Positive Rate)</strong>
<ul>
<li>The probability that the test is positive when the disease is present</li>
<li>
<strong>Formula:</strong> <span class="math inline">\(\frac{\text{True Positives}}{\text{True Positives} + \text{False Negatives}}\)</span>
</li>
<li>
<strong>Significance:</strong> A test with high sensitivity (e.g., 99%) has very few False Negatives. It is an excellent “Screening” test because you can trust a Negative result to rule out disease</li>
</ul>
</li>
<li>
<strong>Specificity (True Negative Rate)</strong>
<ul>
<li>The probability that the test is negative when the disease is absent</li>
<li>
<strong>Formula:</strong> <span class="math inline">\(\frac{\text{True Negatives}}{\text{True Negatives} + \text{False Positives}}\)</span>
</li>
<li>
<strong>Significance:</strong> A test with high specificity has very few False Positives. It is an excellent “Confirmatory” test</li>
</ul>
</li>
<li>
<strong>Predictive Values (PPV and NPV)</strong>
<ul>
<li>Unlike Sensitivity/Specificity, which are intrinsic properties of the test, Predictive Values are dependent on the <strong>Prevalence:</strong> of the disease in the population tested</li>
<li>
<strong>Positive Predictive Value (PPV):</strong> If the test is positive, how likely is it that the patient actually has the disease? In a low-prevalence environment (e.g., screening asymptomatic people for a rare genetic disorder), the PPV drops, and False Positives constitute a higher percentage of the “Positive” results</li>
<li>
<strong>Negative Predictive Value (NPV):</strong> If the test is negative, how likely is it that the patient is healthy?</li>
</ul>
</li>
</ul>
</div>
<div id="method-validation-statistics-analytical-performance" class="section level2 unnumbered">
<h2>Method Validation Statistics (Analytical Performance)<a class="anchor" aria-label="anchor" href="#method-validation-statistics-analytical-performance"><i class="fas fa-link"></i></a>
</h2>
<p>For a test to be approved for clinical use (especially Laboratory Developed Tests - LDTs), statistical experiments must prove its analytical robustness. This involves Linear Regression and Limit determination</p>
<ul>
<li>
<strong>Linear Regression (<span class="math inline">\(y = mx + b\)</span>)</strong>
<ul>
<li>Used during “Accuracy” or “Method Comparison” studies. The results of the New Method (<span class="math inline">\(y\)</span>) are plotted against the Reference Method (<span class="math inline">\(x\)</span>)</li>
<li>
<strong>Correlation Coefficient (<span class="math inline">\(R\)</span>):</strong> Measures the strength of the relationship. An <span class="math inline">\(R\)</span>-value <span class="math inline">\(&gt; 0.975\)</span> indicates excellent agreement</li>
<li>
<strong>Slope (<span class="math inline">\(m\)</span>):</strong> Indicates Proportional Error. Ideally, the slope is 1.0. A slope of 1.1 suggests the new method runs 10% higher than the old method consistently</li>
<li>
<strong>Y-Intercept (<span class="math inline">\(b\)</span>):</strong> Indicates Constant Error. Ideally, the intercept is 0. If <span class="math inline">\(b = 5\)</span>, the new method consistently adds 5 units to every result regardless of concentration</li>
</ul>
</li>
<li>
<strong>Limit of Detection (LOD)</strong>
<ul>
<li>The lowest concentration of analyte that can be detected (distinguished from zero) with 95% confidence. This is the definition of <strong>Analytical Sensitivity</strong>
</li>
<li>
<strong>Probit Analysis:</strong> To determine LOD, the lab runs serial dilutions of a sample. The LOD is the concentration where 19 out of 20 replicates (95%) return a “Positive” result. This is critical for PCR assays detecting low-level pathogens (e.g., Meningitis)</li>
</ul>
</li>
<li>
<strong>Linearity (<span class="math inline">\(R^2\)</span>)</strong>
<ul>
<li>For quantitative assays (Viral Loads), the lab plots observed values against expected values. The Coefficient of Determination (<span class="math inline">\(R^2\)</span>) measures how well the data fits a straight line. An <span class="math inline">\(R^2\)</span> of <span class="math inline">\(&gt;0.99\)</span> proves the assay is linear across the Analytical Measurement Range (AMR)</li>
</ul>
</li>
</ul>
</div>
<div id="process-control-qc-rules-errors" class="section level2 unnumbered">
<h2>Process Control (QC Rules &amp; Errors)<a class="anchor" aria-label="anchor" href="#process-control-qc-rules-errors"><i class="fas fa-link"></i></a>
</h2>
<p>Once a test is live, statistical rules monitor stability. The Levey-Jennings (L-J) chart plots QC values over time against the Mean and SD. <strong>Westgard Rules</strong> are statistical logic gates used to accept or reject a run based on probability</p>
<ul>
<li>
<strong>Random Error (Precision Issues)</strong>
<ul>
<li>Unpredictable variations affecting individual samples</li>
<li>
<strong>1-3s Rule:</strong> A single QC point exceeds 3 Standard Deviations from the mean. Statistically, this happens by chance only 0.3% of the time. It is a “Rejection” rule indicating a significant random error (e.g., a bubble in the PCR well or a pipetting error)</li>
<li>
<strong>R-4s Rule:</strong> The difference between two controls within the same run exceeds 4 SD (e.g., High Control is +2SD and Low Control is -2SD). This indicates severe imprecision</li>
</ul>
</li>
<li>
<strong>Systematic Error (Accuracy/Bias Issues)</strong>
<ul>
<li>A trend or shift affecting all samples equally</li>
<li>
<strong>2-2s Rule:</strong> Two consecutive QC points exceed 2 SD on the same side of the mean. This indicates a “Shift” or “Trend.”</li>
<li>
<strong>Trend:</strong> Gradual drift (e.g., 7 consecutive points moving upward). Causes: Aging reagents, degrading probes, or lamp dimming in the fluorometer</li>
<li>
<strong>Shift:</strong> Abrupt change. Causes: New lot number of reagents, new calibration, or major instrument maintenance</li>
</ul>
</li>
</ul>
</div>
<div id="six-sigma-metrics-total-quality-management" class="section level2 unnumbered">
<h2>Six Sigma Metrics (Total Quality Management)<a class="anchor" aria-label="anchor" href="#six-sigma-metrics-total-quality-management"><i class="fas fa-link"></i></a>
</h2>
<p>Advanced laboratory administration utilizes Six Sigma concepts to quantify the “Total Error” of an assay and optimize QC frequency. This approach combines Precision and Accuracy into a single metric</p>
<ul>
<li>
<strong>Total Allowable Error (TEa)</strong>
<ul>
<li>The maximum amount of error allowed for a specific analyte before the result compromises patient care. These limits are defined by CLIA or CAP (e.g., Viral Load must be within <span class="math inline">\(\pm 0.5 \log_{10}\)</span> of the target)</li>
</ul>
</li>
<li>
<strong>The Sigma Metric Equation</strong>
<ul>
<li>
<span class="math inline">\(\text{Sigma} (\sigma) = \frac{\text{TEa} - \text{Bias}}{\text{SD}}\)</span> (or <span class="math inline">\(\%CV\)</span>)</li>
<li>
<strong>Interpretation:</strong> The Sigma metric tells you how many Standard Deviations fit between your mean and the allowable error limit</li>
<li>
<strong>Low Sigma (&lt; 3):</strong> The assay is unstable. Frequent QC is required (e.g., every 4 hours) to detect errors</li>
<li>
<strong>High Sigma (&gt; 6):</strong> “World Class” performance. The assay is so precise and accurate that it almost never produces a clinically significant error. The lab can reduce QC frequency and “relax” Westgard rules (e.g., ignore the 1-2s warning rule) to save money without risking patient safety</li>
</ul>
</li>
</ul>
</div>
<div id="molecular-specific-statistical-considerations" class="section level2 unnumbered">
<h2>Molecular-Specific Statistical Considerations<a class="anchor" aria-label="anchor" href="#molecular-specific-statistical-considerations"><i class="fas fa-link"></i></a>
</h2>
<p>Molecular biology introduces unique statistical challenges regarding logarithmic data and Cycle Thresholds</p>
<ul>
<li>
<strong>Logarithmic Transformation (<span class="math inline">\(\log_{10}\)</span>)</strong>
<ul>
<li>Quantitative molecular data (Viral Loads) is rarely normally distributed; it is skewed. We convert raw copies/mL to <span class="math inline">\(\log_{10}\)</span> values for statistical analysis</li>
<li>
<strong>Clinical Significance:</strong> In molecular virology, a variation of less than <strong>0.3 log</strong> (approx. 2-fold) is usually considered inherent biological/statistical noise. A change is often not deemed clinically significant unless it exceeds <strong>0.5 log</strong> (approx. 3-fold). This statistical reality guides how physicians interpret rising or falling viral loads</li>
</ul>
</li>
<li>
<strong>Ct Value Analysis</strong>
<ul>
<li>In Real-Time PCR, the Ct (Cycle Threshold) is inversely proportional to the amount of target nucleic acid</li>
<li>
<strong>QC Monitoring:</strong> Labs track the mean Ct values of Positive Controls. A “Shift” in the mean Ct value (e.g., moving from 24.0 to 26.5) indicates a loss of sensitivity (enzyme degradation) even if the Qualitative result is still “Positive.” This statistical tracking serves as an early warning system before the assay fails completely</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="quality-management-1.html">Quality Management</a></div>
<div class="next"><a href="assay-method-instrument-selection.html">Assay, Method, &amp; Instrument Selection</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#advanced-statistical-analysis-1">Advanced Statistical Analysis</a></li>
<li><a class="nav-link" href="#descriptive-statistics-precision-monitoring">Descriptive Statistics &amp; Precision Monitoring</a></li>
<li><a class="nav-link" href="#diagnostic-accuracy-clinical-performance">Diagnostic Accuracy (Clinical Performance)</a></li>
<li><a class="nav-link" href="#method-validation-statistics-analytical-performance">Method Validation Statistics (Analytical Performance)</a></li>
<li><a class="nav-link" href="#process-control-qc-rules-errors">Process Control (QC Rules &amp; Errors)</a></li>
<li><a class="nav-link" href="#six-sigma-metrics-total-quality-management">Six Sigma Metrics (Total Quality Management)</a></li>
<li><a class="nav-link" href="#molecular-specific-statistical-considerations">Molecular-Specific Statistical Considerations</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/04-F-04-a.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/04-F-04-a.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
