<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Array Technology | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="This technology has dramatically scaled up our ability to ask questions of a patient’s DNA or RNA. If a single PCR is like making one phone call to ask one specific question, array technology is...">
<meta name="generator" content="bookdown 0.43 with bs4_book()">
<meta property="og:title" content="Array Technology | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="This technology has dramatically scaled up our ability to ask questions of a patient’s DNA or RNA. If a single PCR is like making one phone call to ask one specific question, array technology is...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Array Technology | MB">
<meta name="twitter:description" content="This technology has dramatically scaled up our ability to ask questions of a patient’s DNA or RNA. If a single PCR is like making one phone call to ask one specific question, array technology is...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-and-detection.html">Separation and Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="active" href="array-technology-1.html">Array Technology</a></li>
<li><a class="" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="array-technology-1" class="section level1 unnumbered">
<h1>Array Technology<a class="anchor" aria-label="anchor" href="#array-technology-1"><i class="fas fa-link"></i></a>
</h1>
<p>This technology has dramatically scaled up our ability to ask questions of a patient’s DNA or RNA. If a single PCR is like making one phone call to ask one specific question, <strong>array technology</strong> is like sending out a massive survey to thousands of recipients all at once and getting all the answers back simultaneously. It’s a powerful tool for high-throughput, parallel analysis</p>
<p>At its heart, array technology is a <strong>multiplex hybridization assay</strong>. The core principle involves immobilizing thousands of known DNA sequences (<strong>probes</strong>) onto a solid surface in an orderly fashion. A patient’s sample (the <strong>target</strong>) is then labeled and washed over this surface. By seeing where the patient’s sample sticks, we can learn thousands of things about it in a single experiment</p>
<p>There are two main “flavors” of this technology that you’ll encounter: the classic microarray and the flexible bead array</p>
<div id="microarrays-the-chip" class="section level2 unnumbered">
<h2>
<strong>Microarrays (The “Chip”)</strong><a class="anchor" aria-label="anchor" href="#microarrays-the-chip"><i class="fas fa-link"></i></a>
</h2>
<p>Think of a microarray as a microscopic grid printed onto a glass slide. Each tiny, invisible spot on this grid has a different, known DNA probe chemically attached to it. The location of the spot tells you what the probe is</p>
<ul>
<li>
<strong>The Setup:</strong> A glass slide, often no bigger than a standard microscope slide, is spotted with tens of thousands to millions of unique DNA probes. The <code>x,y</code> coordinate of each spot is meticulously tracked in a database</li>
<li>
<strong>How it Works:</strong> The patient’s DNA or RNA (the target) is isolated, fragmented, and labeled with a fluorescent dye. This labeled sample is then incubated with the microarray slide. The patient’s DNA fragments will hybridize, or “stick,” only to the spots containing their complementary probe sequence. After washing away any non-specific binding, the slide is scanned with a laser. The scanner detects the fluorescence at each spot, and the intensity of the light is proportional to the amount of patient DNA that bound there</li>
<li>
<strong>The Result:</strong> A computer image of the slide with a grid of glowing dots of varying intensities. Since the identity of the probe at every coordinate is known, the software can quickly generate a massive report of all the sequences present in the patient’s sample</li>
</ul>
</div>
<div id="bead-arrays-the-liquid-array" class="section level2 unnumbered">
<h2>
<strong>Bead Arrays (The “Liquid Array”)</strong><a class="anchor" aria-label="anchor" href="#bead-arrays-the-liquid-array"><i class="fas fa-link"></i></a>
</h2>
<p>Bead array technology, most famously represented by Luminex xMAP technology, takes the grid concept and puts it into a liquid format. Instead of a fixed grid on a slide, the probes are attached to microscopic beads (microspheres)</p>
<ul>
<li>
<strong>The Setup:</strong> The system uses up to 500 different sets of tiny beads. Each set is internally dyed with a unique combination of red and infrared fluorophores, giving each bead set a specific spectral “address” or color code. All the beads within one color-coded set are coated with the same specific probe sequence</li>
<li>
<strong>How it Works:</strong> The different sets of probe-coated beads are all mixed together in a single tube. The patient’s labeled sample is added to this mixture, and hybridization occurs in suspension. The entire mixture is then drawn into a specialized flow cytometer</li>
<li>
<strong>Detection:</strong> As each individual bead flows single-file past two lasers:
<ol style="list-style-type: decimal">
<li>The <strong>Classification Laser</strong> hits the bead and reads its internal color code, identifying <em>which probe</em> is on that bead (e.g., “this is a bead for the <em>CYP2C19</em> *2 allele”)</li>
<li>The <strong>Reporter Laser</strong> excites the fluorescent label on the patient’s sample that has bound to the bead’s surface, measuring <em>how much</em> has bound</li>
</ol>
</li>
<li>
<strong>The Result:</strong> The instrument can analyze thousands of beads per second, rapidly building a report on the presence and quantity of hundreds of different targets in a single well of a 96-well plate</li>
</ul>
</div>
<div id="general-workflow-clinical-applications" class="section level2 unnumbered">
<h2>
<strong>General Workflow &amp; Clinical Applications</strong><a class="anchor" aria-label="anchor" href="#general-workflow-clinical-applications"><i class="fas fa-link"></i></a>
</h2>
<p>Regardless of the platform, the core applications are similar and highlight the power of multiplexing</p>
<ol style="list-style-type: decimal">
<li>
<strong>Sample Preparation &amp; Labeling</strong> Patient DNA/RNA is isolated, sometimes amplified via PCR, and labeled with a fluorescent tag (often biotin, which is then detected by a fluorescent streptavidin conjugate)</li>
<li>
<strong>Hybridization</strong> The labeled sample is incubated with the array (chip or bead mix)</li>
<li>
<strong>Washing</strong> Unbound sample is washed away to reduce background noise</li>
<li>
<strong>Signal Detection</strong> The array is read by a scanner or flow cytometer</li>
<li>
<strong>Data Analysis</strong> Sophisticated software converts the raw signal intensities into a clinical result</li>
</ol>
<div id="major-clinical-uses" class="section level3 unnumbered">
<h3>
<strong>Major Clinical Uses</strong><a class="anchor" aria-label="anchor" href="#major-clinical-uses"><i class="fas fa-link"></i></a>
</h3>
<ul>
<li>
<strong>Pharmacogenomic (PGx) Testing:</strong> This is a perfect application. A single array can simultaneously test for dozens of key SNPs in genes involved in drug metabolism (e.g., <em>CYP2C19</em>, <em>CYP2D6</em>, <em>VKORC1</em>, <em>TPMT</em>), providing a comprehensive drug-response profile for a patient in one test</li>
<li>
<strong>Cytogenomics (Array CGH):</strong> <strong>Array Comparative Genomic Hybridization</strong> is a major diagnostic tool for detecting <strong>copy number variations (CNVs)</strong>—microdeletions and microduplications that are too small to be seen on a traditional karyotype
<ul>
<li>
<strong>How it works:</strong> The patient’s DNA and a normal reference DNA sample are labeled with two different colored dyes (e.g., patient = green, reference = red). They are mixed and co-hybridized to a microarray containing probes spanning the entire genome</li>
<li>
<strong>Interpretation:</strong> If a spot on the array shows a <strong>yellow</strong> color (equal parts red and green), the patient has a normal copy number at that location. A <strong>green</strong> color means the patient has more DNA than the reference (a duplication/gain). A <strong>red</strong> color means the patient has less DNA than the reference (a deletion/loss). This is a primary test for developmental delay and birth defects</li>
</ul>
</li>
<li>
<strong>Gene Expression Profiling:</strong> Microarrays can measure the expression levels of thousands of genes at once by analyzing labeled cDNA made from the patient’s mRNA. In oncology, this can help classify tumors into different subtypes (e.g., Oncotype DX for breast cancer), which can predict prognosis and guide treatment decisions</li>
<li>
<strong>Multiplex Pathogen Detection:</strong> Arrays can be designed with probes for dozens of different respiratory viruses or bacteria, allowing for a broad infectious disease panel from a single patient swab</li>
<li>
<strong>Multiplex Immunoassays:</strong> Bead arrays are especially powerful for non-DNA applications. They can be coated with antigens to detect dozens of different antibodies at once (e.g., HLA antibody screening for transplant patients) or coated with antibodies to measure dozens of different proteins/cytokines from a single drop of blood</li>
</ul>
</div>
</div>
<div id="key-terms-41" class="section level2 unnumbered">
<h2>
<strong>Key Terms</strong><a class="anchor" aria-label="anchor" href="#key-terms-41"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>Array Technology:</strong> A high-throughput platform that allows for the simultaneous measurement of thousands of different biological molecules (like DNA, RNA, or protein) by using a grid of immobilized probes</li>
<li>
<strong>Probe:</strong> The known, single-stranded molecule (e.g., an oligonucleotide) that is fixed to a specific location on the array surface and is designed to capture a specific target molecule</li>
<li>
<strong>Hybridization:</strong> The process by which the labeled target molecules from the patient sample bind to their complementary probes on the array surface</li>
<li>
<strong>Multiplexing:</strong> The ability to perform many different tests or assays simultaneously in a single reaction, which is the core advantage of array technology</li>
<li>
<strong>Microarray:</strong> An array format where probes are spotted in a high-density grid onto a solid surface, typically a glass slide. The location (<code>x,y</code> coordinate) of a signal identifies the target</li>
<li>
<strong>Bead Array:</strong> An array format where probes are attached to microscopic, color-coded beads. A flow cytometry-based instrument identifies the bead by its color and measures the signal on its surface</li>
<li>
<strong>Array CGH (Comparative Genomic Hybridization):</strong> A microarray-based technique used to detect copy number variations (gains and losses of DNA) across the genome by comparing a differentially labeled patient sample to a reference sample</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></div>
<div class="next"><a href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#array-technology-1">Array Technology</a></li>
<li><a class="nav-link" href="#microarrays-the-chip">Microarrays (The “Chip”)</a></li>
<li><a class="nav-link" href="#bead-arrays-the-liquid-array">Bead Arrays (The “Liquid Array”)</a></li>
<li>
<a class="nav-link" href="#general-workflow-clinical-applications">General Workflow &amp; Clinical Applications</a><ul class="nav navbar-nav"><li><a class="nav-link" href="#major-clinical-uses">Major Clinical Uses</a></li></ul>
</li>
<li><a class="nav-link" href="#key-terms-41">Key Terms</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/02-E-06.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/02-E-06.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
