<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Assay Troubleshooting | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="Troubleshooting in the molecular laboratory is the systematic process of identifying the root cause of an assay failure or an unexpected result. Unlike chemical assays, where an instrument flag...">
<meta name="generator" content="bookdown 0.45 with bs4_book()">
<meta property="og:title" content="Assay Troubleshooting | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="Troubleshooting in the molecular laboratory is the systematic process of identifying the root cause of an assay failure or an unexpected result. Unlike chemical assays, where an instrument flag...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Assay Troubleshooting | MB">
<meta name="twitter:description" content="Troubleshooting in the molecular laboratory is the systematic process of identifying the root cause of an assay failure or an unexpected result. Unlike chemical assays, where an instrument flag...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-detection.html">Separation &amp; Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="" href="array-technology-1.html">Array Technology</a></li>
<li><a class="" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
<li class="book-part">Laboratory Operations</li>
<li><a class="" href="contamination.html">Contamination</a></li>
<li><a class="" href="prevention-1.html">Prevention</a></li>
<li><a class="" href="monitoring-detection-1.html">Monitoring &amp; Detection</a></li>
<li><a class="" href="elimination-1.html">Elimination</a></li>
<li><a class="" href="quality-assurance.html">Quality Assurance</a></li>
<li><a class="" href="specimen-handling.html">Specimen Handling</a></li>
<li><a class="" href="specimen-evaluation.html">Specimen Evaluation</a></li>
<li><a class="" href="nucleic-acid-evaluation.html">Nucleic Acid Evaluation</a></li>
<li><a class="" href="reagents.html">Reagents</a></li>
<li><a class="" href="assays.html">Assays</a></li>
<li><a class="active" href="assay-troubleshooting-1.html">Assay Troubleshooting</a></li>
<li><a class="" href="result-calculation-interpretation-reporting-1.html">Result Calculation, Interpretation, &amp; Reporting</a></li>
<li><a class="" href="qc-pt.html">QC &amp; PT</a></li>
<li><a class="" href="assay-controls.html">Assay Controls</a></li>
<li><a class="" href="proficiency-testing.html">Proficiency Testing</a></li>
<li><a class="" href="equipment-instrumentation-1.html">Equipment &amp; Instrumentation</a></li>
<li><a class="" href="guidelines-regulations.html">Guidelines &amp; Regulations</a></li>
<li><a class="" href="test-system-categories-1.html">Test System Categories</a></li>
<li><a class="" href="regulations-standards-1.html">Regulations &amp; Standards</a></li>
<li><a class="" href="personnel.html">Personnel</a></li>
<li><a class="" href="continuing-education-1.html">Continuing Education</a></li>
<li><a class="" href="competency.html">Competency</a></li>
<li><a class="" href="safety.html">Safety</a></li>
<li><a class="" href="hazardous-materials.html">Hazardous Materials</a></li>
<li><a class="" href="biological.html">Biological</a></li>
<li><a class="" href="chemical.html">Chemical</a></li>
<li><a class="" href="laboratory-administration.html">Laboratory Administration</a></li>
<li><a class="" href="financial-management-1.html">Financial Management</a></li>
<li><a class="" href="budgets-1.html">Budgets</a></li>
<li><a class="" href="capital-equipment.html">Capital Equipment</a></li>
<li><a class="" href="cost-analysis-reimbursement-1.html">Cost Analysis &amp; Reimbursement</a></li>
<li><a class="" href="purchasing-inventory-1.html">Purchasing &amp; Inventory</a></li>
<li><a class="" href="operations-management-1.html">Operations Management</a></li>
<li><a class="" href="laboratory-information-system.html">Laboratory Information System</a></li>
<li><a class="" href="facilities-management-1.html">Facilities Management</a></li>
<li><a class="" href="intrainterdepartmental-relations-1.html">Intra/Interdepartmental Relations</a></li>
<li><a class="" href="personnel-management-1.html">Personnel Management</a></li>
<li><a class="" href="motivation-1.html">Motivation</a></li>
<li><a class="" href="staffing-productivity-1.html">Staffing &amp; Productivity</a></li>
<li><a class="" href="counseling-disciplinary-action-1.html">Counseling &amp; Disciplinary Action</a></li>
<li><a class="" href="quality-management-1.html">Quality Management</a></li>
<li><a class="" href="advanced-statistical-analysis-1.html">Advanced Statistical Analysis</a></li>
<li><a class="" href="assay-method-instrument-selection.html">Assay, Method, &amp; Instrument Selection</a></li>
<li><a class="" href="assay-validation-verification.html">Assay Validation &amp; Verification</a></li>
<li><a class="" href="quantitative-calculations-1.html">Quantitative Calculations</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="assay-troubleshooting-1" class="section level1 unnumbered">
<h1>Assay Troubleshooting<a class="anchor" aria-label="anchor" href="#assay-troubleshooting-1"><i class="fas fa-link"></i></a>
</h1>
<p>Troubleshooting in the molecular laboratory is the systematic process of identifying the root cause of an assay failure or an unexpected result. Unlike chemical assays, where an instrument flag often identifies the specific error, molecular diagnostics relies heavily on the interpretation of experimental controls (Internal Controls, Positive Controls, and No Template Controls) to triangulate the source of the problem. Issues generally fall into three categories: False Negatives (reaction failure), False Positives (contamination), and Quantitative/Efficiency failures</p>
<div id="scenario-1-no-amplification-false-negatives" class="section level2 unnumbered">
<h2>Scenario 1: No Amplification (False Negatives)<a class="anchor" aria-label="anchor" href="#scenario-1-no-amplification-false-negatives"><i class="fas fa-link"></i></a>
</h2>
<p>This is the most common failure mode, characterized by a lack of fluorescence signal or band presence in a sample known or suspected to be positive. The primary challenge is distinguishing between a true negative patient and a failed reaction</p>
<ul>
<li>
<strong>Enzymatic Inhibition</strong>
<ul>
<li>
<strong>Symptoms:</strong> The Patient sample is negative, and the <strong>Internal Control (IC) fails</strong> (no amplification), but the external Positive Control works perfectly</li>
<li>
<strong>Root Cause:</strong> Interfering substances from the clinical matrix co-extracted with the DNA/RNA. Common inhibitors include Heme (blood), Heparin (green top tubes), crystals (urine), or residual ethanol/salt from the extraction wash steps</li>
<li>
<strong>Corrective Action:</strong> Dilute the extracted sample (1:10) and re-run. Dilution reduces the concentration of the inhibitor enough to allow the enzyme to work, often without diluting the DNA below the limit of detection. Alternatively, repeat the extraction</li>
</ul>
</li>
<li>
<strong>Reagent Failure</strong>
<ul>
<li>
<strong>Symptoms:</strong> Patient samples, Internal Controls, and the <strong>Positive Control</strong> all fail. The run is effectively “flat”</li>
<li>
<strong>Root Cause:</strong> The Master Mix components have degraded or were omitted. This is often caused by expired enzymes, leaving reagents at room temperature too long, or repeated freeze-thaw cycles destroying dNTPs or primer stability</li>
<li>
<strong>Corrective Action:</strong> Discard the current aliquot of reagents. Thaw a fresh lot of Master Mix/Primers and repeat the entire run</li>
</ul>
</li>
<li>
<strong>Extraction Failure</strong>
<ul>
<li>
<strong>Symptoms:</strong> Patient sample is negative, IC fails, but the Positive Control works. However, unlike inhibition, a 1:10 dilution does not fix the problem</li>
<li>
<strong>Root Cause:</strong> The DNA/RNA was simply not recovered. This can happen if the lysis buffer precipitated, the spin column was clogged, or the elution buffer was omitted</li>
<li>
<strong>Corrective Action:</strong> The sample must be re-extracted from the original primary specimen</li>
</ul>
</li>
</ul>
</div>
<div id="scenario-2-unexpected-amplification-false-positives" class="section level2 unnumbered">
<h2>Scenario 2: Unexpected Amplification (False Positives)<a class="anchor" aria-label="anchor" href="#scenario-2-unexpected-amplification-false-positives"><i class="fas fa-link"></i></a>
</h2>
<p>A false positive occurs when a signal is generated in a sample that should be negative. This is critical because it can lead to incorrect diagnoses and unnecessary treatment</p>
<ul>
<li>
<strong>Contamination (The “Sentinel” Failure)</strong>
<ul>
<li>
<strong>Symptoms:</strong> Amplification is observed in the <strong>No Template Control (NTC)</strong> or the <strong>Negative Extraction Control</strong>
</li>
<li>
<strong>Root Cause:</strong> Introduction of amplicon (carryover) or environmental genomic DNA into the reagents</li>
<li>
<strong>Corrective Action:</strong> The entire run is invalid. Do not report any positive patient results. The laboratory must perform a decontamination “deep clean,” discard open reagents, and repeat the run with fresh reagents</li>
</ul>
</li>
<li>
<strong>Non-Specific Amplification (Primer-Dimers)</strong>
<ul>
<li>
<strong>Symptoms:</strong> In SYBR Green assays, a signal is detected in the NTC, but the <strong>Melt Curve</strong> shows a peak at a much lower temperature (Tm) than the specific target</li>
<li>
<strong>Root Cause:</strong> Primers binding to each other rather than the template, usually due to poor primer design or low annealing temperatures. This creates a small double-stranded product that binds dye</li>
<li>
<strong>Corrective Action:</strong> Analyze the Melt Curve. If the Tm is distinct from the pathogen target, the result can be considered a “negative” for the target, but the assay design may need optimization</li>
</ul>
</li>
<li>
<strong>Cross-Talk (Spectral Bleed-Through)</strong>
<ul>
<li>
<strong>Symptoms:</strong> A sample has a very strong signal in Channel A (e.g., FAM) and a weak, late signal in Channel B (e.g., VIC), even though the patient is negative for Target B</li>
<li>
<strong>Root Cause:</strong> The fluorescence from the dominant dye is “bleeding” into the detector for the second dye due to overlapping emission spectra or failed color compensation calibration</li>
<li>
<strong>Corrective Action:</strong> Recalibrate the instrument (Color Compensation/Pure Dye calibration). If reporting, review the raw spectral data; if the signal in Channel B perfectly mirrors the rise of Channel A, it is likely artifactual</li>
</ul>
</li>
</ul>
</div>
<div id="scenario-3-poor-quantitative-performance" class="section level2 unnumbered">
<h2>Scenario 3: Poor Quantitative Performance<a class="anchor" aria-label="anchor" href="#scenario-3-poor-quantitative-performance"><i class="fas fa-link"></i></a>
</h2>
<p>In quantitative assays (e.g., Viral Loads), the reaction may work, but the data quality is poor, leading to inaccurate quantification</p>
<ul>
<li>
<strong>Baseline/Threshold Issues</strong>
<ul>
<li>
<strong>Symptoms:</strong> The amplification curve looks “noisy” or the software sets the Crossing Threshold (Ct) too early, resulting in a calculated concentration that is impossibly high</li>
<li>
<strong>Root Cause:</strong> Automatic software algorithms occasionally misinterpret background noise as the linear growth phase</li>
<li>
<strong>Corrective Action:</strong> Switch to “Manual Analysis.” Manually adjust the threshold bar so that it sits above the background noise but within the exponential phase (the straight part) of the logarithmic amplification plot</li>
</ul>
</li>
<li>
<strong>Low Reaction Efficiency</strong>
<ul>
<li>
<strong>Symptoms:</strong> Late Ct values for samples that should be strong positives, or a Standard Curve with a slope outside the acceptable range (-3.1 to -3.6)</li>
<li>
<strong>Root Cause:</strong> Suboptimal reaction conditions. This could be due to pipetting errors in the standards, degraded polymerase, or poor thermal cycling (calibration drift in the heating block)</li>
<li>
<strong>Corrective Action:</strong> Check the <span class="math inline">\(R^2\)</span> value of the standard curve. If <span class="math inline">\(&lt;0.98\)</span>, omit the outlier standard point and recalculate. If the slope is still poor, recalibrate the thermocycler or replace reagents</li>
</ul>
</li>
</ul>
</div>
<div id="the-troubleshooting-algorithm" class="section level2 unnumbered">
<h2>The Troubleshooting Algorithm<a class="anchor" aria-label="anchor" href="#the-troubleshooting-algorithm"><i class="fas fa-link"></i></a>
</h2>
<p>When a run fails, the laboratory scientist should follow a standard logic path to identify the source. This is often documented as a “Corrective Action”</p>
<ol style="list-style-type: decimal">
<li>
<strong>Check the NTC:</strong> Is it positive?
<ul>
<li>
<strong>Yes:</strong> Contamination. Stop. Clean. Repeat</li>
<li>
<strong>No:</strong> Proceed to step 2</li>
</ul>
</li>
<li>
<strong>Check the Positive Control:</strong> Did it amplify within the expected range?
<ul>
<li>
<strong>No:</strong> Reagent or Thermal Cycler failure. Repeat run with new reagents</li>
<li>
<strong>Yes:</strong> The reagents are fine. Proceed to step 3</li>
</ul>
</li>
<li>
<strong>Check the Internal Control (IC):</strong> Did it amplify in the patient samples?
<ul>
<li>
<strong>No:</strong> Extraction failure or Inhibition. Dilute 1:10 or Re-extract</li>
<li>
<strong>Yes:</strong> The sample is truly Negative</li>
</ul>
</li>
<li>
<strong>Check the Curve Shape:</strong> Is it Sigmoidal (S-shaped)?
<ul>
<li>
<strong>No:</strong> If the curve is linear, jagged, or dips, it is likely artifact/noise. Do not report</li>
</ul>
</li>
</ol>
</div>
<div id="documentation-corrective-action-reports" class="section level2 unnumbered">
<h2>Documentation (Corrective Action Reports)<a class="anchor" aria-label="anchor" href="#documentation-corrective-action-reports"><i class="fas fa-link"></i></a>
</h2>
<p>In a CLIA-accredited laboratory, troubleshooting is not complete until it is documented. If an assay fails, a <strong>Corrective Action Report (CAR)</strong> must be filed</p>
<ul>
<li>
<strong>Problem Description:</strong> Clearly state what happened (e.g., “Run 123 failed; Positive Control did not amplify”)</li>
<li>
<strong>Investigation:</strong> Detail the steps taken (e.g., “Checked expiration dates - all valid. Reviewed thermocycler logs - no errors”)</li>
<li>
<strong>Root Cause</strong> Identify the source (e.g., “Master Mix lot #554 was left on the bench overnight by previous shift”)</li>
<li>
<strong>Resolution</strong> State the fix (e.g., “Thawed new Master Mix lot #555. Repeated run. QC passed”)</li>
<li>
<strong>Sign-off:</strong> The report must be reviewed and signed by the Laboratory Director or Technical Supervisor</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="assays.html">Assays</a></div>
<div class="next"><a href="result-calculation-interpretation-reporting-1.html">Result Calculation, Interpretation, &amp; Reporting</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#assay-troubleshooting-1">Assay Troubleshooting</a></li>
<li><a class="nav-link" href="#scenario-1-no-amplification-false-negatives">Scenario 1: No Amplification (False Negatives)</a></li>
<li><a class="nav-link" href="#scenario-2-unexpected-amplification-false-positives">Scenario 2: Unexpected Amplification (False Positives)</a></li>
<li><a class="nav-link" href="#scenario-3-poor-quantitative-performance">Scenario 3: Poor Quantitative Performance</a></li>
<li><a class="nav-link" href="#the-troubleshooting-algorithm">The Troubleshooting Algorithm</a></li>
<li><a class="nav-link" href="#documentation-corrective-action-reports">Documentation (Corrective Action Reports)</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/04-B-04.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/04-B-04.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
