<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Bioinformatics | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="Bioinformatics lives at the intersection of biology, computer science, and statistics. But let’s demystify it. In the clinical molecular lab, bioinformatics isn’t about writing code from scratch;...">
<meta name="generator" content="bookdown 0.43 with bs4_book()">
<meta property="og:title" content="Bioinformatics | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="Bioinformatics lives at the intersection of biology, computer science, and statistics. But let’s demystify it. In the clinical molecular lab, bioinformatics isn’t about writing code from scratch;...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Bioinformatics | MB">
<meta name="twitter:description" content="Bioinformatics lives at the intersection of biology, computer science, and statistics. But let’s demystify it. In the clinical molecular lab, bioinformatics isn’t about writing code from scratch;...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-and-detection.html">Separation and Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="active" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="" href="array-technology-1.html">Array Technology</a></li>
<li><a class="" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="bioinformatics" class="section level1 unnumbered">
<h1>Bioinformatics<a class="anchor" aria-label="anchor" href="#bioinformatics"><i class="fas fa-link"></i></a>
</h1>
<p><strong>Bioinformatics</strong> lives at the intersection of biology, computer science, and statistics. But let’s demystify it. In the clinical molecular lab, bioinformatics isn’t about writing code from scratch; it’s about using powerful software tools to transform the raw, chaotic output from a sequencer into a clear, accurate, and clinically actionable patient report</p>
<p>Think of it this way: a DNA sequencer produces the equivalent of a massive, unreadable text file containing millions or billions of A’s, T’s, C’s, and G’s. Bioinformatics is the set of tools and processes we use to read that file, find the spelling mistakes (variants), and look up what those mistakes mean in the medical dictionary</p>
<div id="the-clinical-bioinformatics-pipeline-a-journey-from-raw-data-to-report" class="section level2 unnumbered">
<h2>
<strong>The Clinical Bioinformatics Pipeline: A Journey from Raw Data to Report</strong><a class="anchor" aria-label="anchor" href="#the-clinical-bioinformatics-pipeline-a-journey-from-raw-data-to-report"><i class="fas fa-link"></i></a>
</h2>
<p>Most clinical sequencing, especially Next-Generation Sequencing (NGS), follows a standardized workflow or “pipeline.” While the specific software may vary, the logical steps are universal</p>
<div id="step-1-primary-analysis-quality-control-qc" class="section level3 unnumbered">
<h3>
<strong>Step 1: Primary Analysis &amp; Quality Control (QC)</strong><a class="anchor" aria-label="anchor" href="#step-1-primary-analysis-quality-control-qc"><i class="fas fa-link"></i></a>
</h3>
<p>This happens right after the sequencing run finishes. The goal is to answer one simple question: <strong>“Is the raw data good enough to analyze?”</strong></p>
<ul>
<li>
<strong>Raw Data Format:</strong> The sequencer spits out raw data in a <strong>FASTQ file</strong>. This file contains two key pieces of information for each short sequence “read”:
<ol style="list-style-type: decimal">
<li>The sequence of bases (e.g., ATTCGGTAC…)</li>
<li>A <strong>Phred Quality Score (Q-score)</strong> for each base, indicating the probability that the base was called correctly. A high Q-score (e.g., Q30) means high confidence; a low score means the sequencer wasn’t sure</li>
</ol>
</li>
<li>
<strong>What We Check:</strong> The bioinformatics pipeline starts by generating QC reports. A technologist is often the first person to review these. We look for:
<ul>
<li>
<strong>High Average Quality Scores:</strong> We want to see that most bases have a Q-score of 30 or higher (meaning a 1 in 1000 chance of being an error)</li>
<li>
<strong>Sufficient Number of Reads:</strong> Did the run produce enough data to adequately cover the genes we’re interested in?</li>
<li>
<strong>Adapter Contamination:</strong> Are there leftover bits of sequencing adapters that need to be trimmed off?</li>
<li>
<strong>Expected Base Composition:</strong> Does the percentage of A, T, C, and G look normal?</li>
</ul>
</li>
</ul>
<p>If the data fails QC, it’s garbage in, garbage out. The sample must be re-prepped or re-sequenced</p>
</div>
<div id="step-2-alignment-mapping" class="section level3 unnumbered">
<h3>
<strong>Step 2: Alignment / Mapping</strong><a class="anchor" aria-label="anchor" href="#step-2-alignment-mapping"><i class="fas fa-link"></i></a>
</h3>
<p>Once we have good quality data, we need to figure out where each of these millions of short reads belongs in the human genome</p>
<ul>
<li>
<strong>The Principle:</strong> This step is like assembling a giant jigsaw puzzle. The short reads from our FASTQ file are the puzzle pieces, and the <strong>human reference genome</strong> (the “normal” sequence, e.g., GRCh38) is the picture on the box cover</li>
<li>
<strong>The Process:</strong> An “aligner” or “mapper” program takes each read and finds its most likely location in the reference genome</li>
<li>
<strong>The Output:</strong> This process generates a <strong>Sequence Alignment/Map (SAM) file</strong>, which is usually compressed into a <strong>Binary Alignment/Map (BAM) file</strong> to save space. A BAM file contains all the reads, now neatly stacked up against the part of the genome where they belong</li>
</ul>
</div>
<div id="step-3-variant-calling" class="section level3 unnumbered">
<h3>
<strong>Step 3: Variant Calling</strong><a class="anchor" aria-label="anchor" href="#step-3-variant-calling"><i class="fas fa-link"></i></a>
</h3>
<p>This is the core discovery step. Now that our reads are aligned, we can systematically scan them to find where the patient’s sequence differs from the reference genome</p>
<ul>
<li>
<strong>The Principle:</strong> The variant caller software looks at each position in the genome. If the reference genome has a “G” at a certain spot, but the majority of the patient’s aligned reads have an “A,” the software “calls” a variant at that position</li>
<li>
<strong>Types of Variants Detected</strong>
<ul>
<li>
<strong>Single Nucleotide Polymorphisms (SNPs) / Single Nucleotide Variants (SNVs):</strong> A change in a single base (e.g., G &gt; A)</li>
<li>
<strong>Insertions/Deletions (Indels):</strong> The addition or removal of one or more bases</li>
</ul>
</li>
<li>
<strong>The Output:</strong> The list of all identified differences is stored in a <strong>Variant Call Format (VCF) file</strong>. This file is essentially a list of every position where the patient’s DNA is different from the reference</li>
</ul>
</div>
<div id="step-4-annotation-interpretation" class="section level3 unnumbered">
<h3>
<strong>Step 4: Annotation &amp; Interpretation</strong><a class="anchor" aria-label="anchor" href="#step-4-annotation-interpretation"><i class="fas fa-link"></i></a>
</h3>
<p>A raw VCF file might contain thousands of variants, but most are benign or common polymorphisms. The next step is to figure out which ones, if any, are clinically important. This is <strong>annotation</strong></p>
<ul>
<li>
<strong>The Principle:</strong> The software takes each variant from the VCF file and cross-references it with massive public and private databases to add context</li>
<li>
<strong>Key Questions Annotation Answers</strong>
<ul>
<li>
<strong>Location:</strong> What gene is this variant in? Is it in an exon (protein-coding region) or an intron?</li>
<li>
<strong>Effect:</strong> What does this variant do to the protein? Is it a <strong>missense</strong> (changes one amino acid), <strong>nonsense</strong> (creates a premature stop codon), <strong>frameshift</strong>, or <strong>silent</strong> mutation?</li>
<li>
<strong>Prevalence:</strong> How common is this variant in the general population? Databases like <strong>gnomAD</strong> tell us the allele frequency. A very common variant is unlikely to be the cause of a rare disease</li>
<li>
<strong>Clinical Significance:</strong> Has this variant been reported before as benign, pathogenic, or of uncertain significance? This information is pulled from databases like <strong>ClinVar</strong>
</li>
<li>
<strong>Functional Prediction:</strong> In silico (computer-based) tools like SIFT and PolyPhen predict whether an amino acid change is likely to be damaging to the protein’s function</li>
</ul>
</li>
</ul>
</div>
<div id="step-5-filtering-review-and-reporting" class="section level3 unnumbered">
<h3>
<strong>Step 5: Filtering, Review, and Reporting</strong><a class="anchor" aria-label="anchor" href="#step-5-filtering-review-and-reporting"><i class="fas fa-link"></i></a>
</h3>
<p>The final step is to filter the massively annotated list down to a handful of variants that are relevant to the patient’s clinical picture</p>
<ul>
<li>
<strong>The Process:</strong> A series of filters are applied. For example, in a cancer panel, we might filter to only show:
<ul>
<li>Variants inside the specific genes on our panel</li>
<li>Variants that are not silent</li>
<li>Variants with an allele frequency below 1% in the population</li>
<li>Variants previously classified as pathogenic or likely pathogenic</li>
</ul>
</li>
<li>
<strong>The Result:</strong> This process yields a short list of potentially significant variants. A trained molecular pathologist or geneticist then reviews this evidence to make a final interpretation, which is then written into the patient’s report</li>
</ul>
</div>
</div>
<div id="the-role-of-the-medical-laboratory-scientist-in-bioinformatics" class="section level2 unnumbered">
<h2>
<strong>The Role of the Medical Laboratory Scientist in Bioinformatics</strong><a class="anchor" aria-label="anchor" href="#the-role-of-the-medical-laboratory-scientist-in-bioinformatics"><i class="fas fa-link"></i></a>
</h2>
<p>As an entry-level MLS, you are not expected to be a bioinformatics programmer. However, you are a crucial user and first-line-of-defense for the bioinformatics pipeline. Your role includes:</p>
<ul>
<li>
<strong>Initiating the Pipeline:</strong> Loading the raw data from the sequencer and starting the analysis workflow on the lab’s server or cloud platform</li>
<li>
<strong>Monitoring QC:</strong> Being the first person to look at the post-run QC reports (e.g., from FastQC or the instrument’s software) to give a “go/no-go” for further analysis</li>
<li>
<strong>Recognizing Failures:</strong> Knowing what a successful pipeline run looks like and being able to identify error messages or reports that indicate a pipeline crash or a problem with the data</li>
<li>
<strong>Data Management:</strong> Understanding the key file types (FASTQ, BAM, VCF) and where they are stored. You are a custodian of patient data</li>
<li>
<strong>Basic Troubleshooting:</strong> Understanding the steps well enough to communicate a problem effectively to a bioinformatics specialist (e.g., “The alignment percentage for sample X was unusually low,” or “The VCF file for this run is empty.”)</li>
</ul>
<p>In short, bioinformatics is the essential final step that gives meaning to all the careful lab work of sample prep and sequencing. It’s the sophisticated engine that turns a sea of letters into a diagnosis</p>
</div>
<div id="key-terms-35" class="section level2 unnumbered">
<h2>
<strong>Key Terms</strong><a class="anchor" aria-label="anchor" href="#key-terms-35"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>Bioinformatics:</strong> The use of computational tools to acquire, store, analyze, and interpret biological data, especially large datasets like genomic sequences</li>
<li>
<strong>FASTQ:</strong> A text-based file format that stores both a nucleotide sequence and its corresponding quality scores. This is the raw output of most NGS platforms</li>
<li>
<strong>Phred Quality Score (Q-score):</strong> A numerical score assigned to each base in a sequence read that represents the estimated accuracy of that base call. Q30 is a common quality benchmark</li>
<li>
<strong>Alignment (or Mapping):</strong> The computational process of determining the original location of a DNA sequence read within a larger reference genome</li>
<li>
<strong>Reference Genome:</strong> A digitally assembled, high-quality “representative” genome sequence for a species (e.g., GRCh38/hg38 for humans) used as a standard for comparison</li>
<li>
<strong>SAM/BAM File:</strong> (Sequence Alignment/Map and its Binary version) File formats that store the results of aligning sequence reads to a reference genome</li>
<li>
<strong>Variant Calling:</strong> The process of identifying positions where the sequenced sample’s DNA differs from the reference genome</li>
<li>
<strong>VCF (Variant Call Format):</strong> A standardized text file format for storing DNA sequence variations (SNPs, indels, etc.)</li>
<li>
<strong>Annotation:</strong> The process of adding relevant biological information to a list of variants, such as the gene affected, the predicted effect on the protein, population frequency, and known clinical significance</li>
<li>
<strong>ClinVar:</strong> A public, FDA-recognized archive that aggregates information about genomic variants and their relationship to human health</li>
<li>
<strong>Allele Frequency:</strong> A measure of how common a specific variant is in a given population</li>
<li>
<strong>Indel:</strong> A type of genetic variant characterized by the insertion or deletion of one or more nucleotides</li>
<li>
<strong>SNP/SNV (Single Nucleotide Polymorphism/Variant):</strong> A variation at a single position in a DNA sequence among individuals</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="other-1.html">Other</a></div>
<div class="next"><a href="other-techniques.html">Other Techniques</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#bioinformatics">Bioinformatics</a></li>
<li>
<a class="nav-link" href="#the-clinical-bioinformatics-pipeline-a-journey-from-raw-data-to-report">The Clinical Bioinformatics Pipeline: A Journey from Raw Data to Report</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#step-1-primary-analysis-quality-control-qc">Step 1: Primary Analysis &amp; Quality Control (QC)</a></li>
<li><a class="nav-link" href="#step-2-alignment-mapping">Step 2: Alignment / Mapping</a></li>
<li><a class="nav-link" href="#step-3-variant-calling">Step 3: Variant Calling</a></li>
<li><a class="nav-link" href="#step-4-annotation-interpretation">Step 4: Annotation &amp; Interpretation</a></li>
<li><a class="nav-link" href="#step-5-filtering-review-and-reporting">Step 5: Filtering, Review, and Reporting</a></li>
</ul>
</li>
<li><a class="nav-link" href="#the-role-of-the-medical-laboratory-scientist-in-bioinformatics">The Role of the Medical Laboratory Scientist in Bioinformatics</a></li>
<li><a class="nav-link" href="#key-terms-35">Key Terms</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/02-D-04.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/02-D-04.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
