<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chemical Structure | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="The Chemical Structure of nucleic acids – built from nucleotides linked by phosphodiester bonds into a directional sugar-phosphate backbone with projecting bases – dictates their function. DNA’s...">
<meta name="generator" content="bookdown 0.43 with bs4_book()">
<meta property="og:title" content="Chemical Structure | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="The Chemical Structure of nucleic acids – built from nucleotides linked by phosphodiester bonds into a directional sugar-phosphate backbone with projecting bases – dictates their function. DNA’s...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chemical Structure | MB">
<meta name="twitter:description" content="The Chemical Structure of nucleic acids – built from nucleotides linked by phosphodiester bonds into a directional sugar-phosphate backbone with projecting bases – dictates their function. DNA’s...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="active" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-and-detection.html">Separation and Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="" href="array-technology-1.html">Array Technology</a></li>
<li><a class="" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="chemical-structure" class="section level1 unnumbered">
<h1>Chemical Structure<a class="anchor" aria-label="anchor" href="#chemical-structure"><i class="fas fa-link"></i></a>
</h1>
<p>The <strong>Chemical Structure</strong> of nucleic acids – built from nucleotides linked by phosphodiester bonds into a directional sugar-phosphate backbone with projecting bases – dictates their function. DNA’s stable, antiparallel double helix with specific A-T/G-C pairing is ideal for information storage. RNA’s single-stranded nature, ribose sugar, and U base allow it to adopt diverse structures for varied roles in gene expression and regulation. Grasping this structure is key to understanding how genetic information is stored, accessed, and analyzed in the clinical laboratory</p>
<p>The chemical structure of nucleic acids ties together sugars and bases. Think of it like building with LEGOs:
* The <strong>sugars</strong> (deoxyribose/ribose) and <strong>phosphates</strong> are the standard connecting bricks that form the long chains
* The <strong>bases</strong> (A, T, G, C, U) are the special feature bricks that stick out and carry the unique information</p>
<div id="the-monomer-nucleotides" class="section level2 unnumbered">
<h2>
<strong>The Monomer: Nucleotides</strong><a class="anchor" aria-label="anchor" href="#the-monomer-nucleotides"><i class="fas fa-link"></i></a>
</h2>
<p>The fundamental repeating unit of both DNA and RNA is the <strong>nucleotide</strong>. Each nucleotide consists of three covalently linked components:</p>
<ul>
<li>
<strong>A Pentose Sugar:</strong> Deoxyribose (in DNA) or Ribose (in RNA). As we discussed, this forms the central scaffold</li>
<li>
<strong>A Nitrogenous Base:</strong> A purine (A or G) or a pyrimidine (C, T, or U). This is attached to the <strong>1’ carbon</strong> of the sugar via an <strong>N-glycosidic bond</strong>. This base holds the genetic code</li>
<li>
<strong>One or More Phosphate Groups:</strong> Usually attached to the <strong>5’ carbon</strong> of the sugar. In the nucleic acid <em>chain</em>, there’s typically one phosphate linking sugars together. Free nucleotides (like ATP) can have one, two, or three phosphate groups (mono-, di-, tri-phosphates)</li>
</ul>
<p><em>(Remember: Base + Sugar = <strong>Nucleoside</strong>. Base + Sugar + Phosphate(s) = <strong>Nucleotide</strong>)</em></p>
</div>
<div id="the-backbone-linking-nucleotides-with-phosphodiester-bonds" class="section level2 unnumbered">
<h2>
<strong>The Backbone: Linking Nucleotides with Phosphodiester Bonds</strong><a class="anchor" aria-label="anchor" href="#the-backbone-linking-nucleotides-with-phosphodiester-bonds"><i class="fas fa-link"></i></a>
</h2>
<p>This is the key to forming the long polymer chains of DNA and RNA. Nucleotides are linked together by <strong>phosphodiester bonds</strong>. Here’s how it works:</p>
<ul>
<li>The <strong>phosphate group</strong> attached to the <strong>5’ carbon</strong> of one nucleotide forms a covalent bond with the <strong>hydroxyl (-OH) group</strong> on the <strong>3’ carbon</strong> of the <em>previous</em> nucleotide’s sugar</li>
<li>This reaction releases a water molecule (pyrophosphate, actually, which then hydrolyzes)</li>
<li>The result is a <strong>sugar-phosphate-sugar-phosphate</strong> chain, forming the <strong>backbone</strong> of the nucleic acid strand</li>
<li>The nitrogenous bases project outwards from this backbone</li>
<li>Crucially, the phosphate groups in the backbone carry a <strong>negative charge</strong> at physiological pH. This makes DNA and RNA acidic molecules (hence “nucleic <em>acid</em>”) and is vital for many lab techniques (like electrophoresis!)</li>
</ul>
</div>
<div id="directionality-the-5-and-3-ends" class="section level2 unnumbered">
<h2>
<strong>Directionality: The 5’ and 3’ Ends</strong><a class="anchor" aria-label="anchor" href="#directionality-the-5-and-3-ends"><i class="fas fa-link"></i></a>
</h2>
<p>The formation of the phosphodiester bond creates an inherent <strong>directionality</strong> in the nucleic acid strand:</p>
<ul>
<li>One end of the strand will have a free phosphate group attached to the 5’ carbon of the terminal sugar. This is called the <strong>5’ end</strong> (five-prime end)</li>
<li>The other end of the strand will have a free hydroxyl (-OH) group on the 3’ carbon of the terminal sugar. This is called the <strong>3’ end</strong> (three-prime end)</li>
<li>By convention, nucleic acid sequences are always written and read from <strong>5’ to 3’</strong>. This directionality is critical for processes like DNA replication and transcription, as enzymes involved (polymerases) typically synthesize new strands in the 5’ to 3’ direction</li>
</ul>
</div>
<div id="dna-structure-the-double-helix" class="section level2 unnumbered">
<h2>
<strong>DNA Structure: The Double Helix</strong><a class="anchor" aria-label="anchor" href="#dna-structure-the-double-helix"><i class="fas fa-link"></i></a>
</h2>
<p>DNA usually exists as a <strong>double helix</strong>, famously described by Watson and Crick. Key features:</p>
<ul>
<li>
<strong>Two Strands:</strong> Two separate polynucleotide chains (sugar-phosphate backbones) are wound around each other</li>
<li>
<strong>Antiparallel:</strong> The two strands run in opposite directions. If one strand runs 5’ to 3’ top-to-bottom, the complementary strand runs 3’ to 5’ top-to-bottom. (Think of two lanes of a highway going in opposite directions)</li>
<li>
<strong>Complementary Base Pairing:</strong> The strands are held together by specific <strong>hydrogen bonds</strong> between the bases projecting inwards:
<ul>
<li>Adenine (A) pairs with Thymine (T) via <strong>two</strong> hydrogen bonds</li>
<li>Guanine (G) pairs with Cytosine (C) via <strong>three</strong> hydrogen bonds</li>
<li>This A=T and G≡C specificity ensures that the sequence of one strand dictates the sequence of the other</li>
</ul>
</li>
<li>
<strong>Helical Shape:</strong> The structure twists into a right-handed helix (most commonly the <strong>B-form DNA</strong>)</li>
<li>
<strong>Major and Minor Grooves:</strong> The twisting creates two grooves on the surface of the helix. The <strong>major groove</strong> is wider, and the <strong>minor groove</strong> is narrower. These grooves are important because they expose the edges of the base pairs, allowing proteins (like transcription factors) to recognize specific DNA sequences without unwinding the helix</li>
<li>
<strong>Stability:</strong> The covalent phosphodiester bonds form a strong, stable backbone. The numerous hydrogen bonds between base pairs, although individually weak, collectively provide significant stability to the double helix. Hydrophobic interactions (base stacking) also contribute</li>
</ul>
</div>
<div id="rna-structure-more-versatile" class="section level2 unnumbered">
<h2>
<strong>RNA Structure: More Versatile</strong><a class="anchor" aria-label="anchor" href="#rna-structure-more-versatile"><i class="fas fa-link"></i></a>
</h2>
<p>RNA differs structurally from DNA in several key ways:</p>
<ul>
<li>
<strong>Sugar:</strong> Contains <strong>ribose</strong> instead of deoxyribose (with that 2’-OH group)</li>
<li>
<strong>Base:</strong> Contains <strong>Uracil (U)</strong> instead of Thymine (T). U pairs with A</li>
<li>
<strong>Single-Stranded:</strong> RNA is typically <strong>single-stranded</strong>, although it can fold back on itself</li>
<li>
<strong>Secondary/Tertiary Structures:</strong> Because it’s usually single-stranded, RNA can fold into complex three-dimensional shapes by forming base pairs (<strong>A=U</strong>, <strong>G≡C</strong>, and sometimes non-canonical pairs like G=U) between different regions of the same molecule. This results in structures like:
<ul>
<li>
<strong>Hairpin loops:</strong> A stem region where the strand is base-paired with itself, capped by a loop of unpaired bases</li>
<li>
<strong>Internal loops and bulges:</strong> Regions of unpaired bases within a double-helical section</li>
<li>
<strong>Pseudoknots:</strong> More complex folding involving loop regions base-pairing with sequences outside the loop</li>
</ul>
</li>
<li>
<strong>Functional Diversity:</strong> These complex structures allow RNA to perform diverse functions beyond just carrying genetic information (like mRNA). Examples include the structural and catalytic roles of ribosomal RNA (rRNA), the adapter function of transfer RNA (tRNA), and the regulatory functions of microRNAs (miRNAs) and long non-coding RNAs (lncRNAs). Some RNAs even act as enzymes (<strong>ribozymes</strong>)</li>
</ul>
</div>
<div id="clinical-laboratory-relevance-2" class="section level2 unnumbered">
<h2>
<strong>Clinical Laboratory Relevance</strong><a class="anchor" aria-label="anchor" href="#clinical-laboratory-relevance-2"><i class="fas fa-link"></i></a>
</h2>
<p>Understanding the chemical structure is fundamental to virtually everything we do in the molecular lab:</p>
<ul>
<li>
<strong>DNA/RNA Extraction:</strong> Methods exploit the chemical properties, like the negative charge (binding to silica matrices or anion exchange resins) and differences in stability (using specific buffers to protect RNA from degradation by its reactive 2’-OH)</li>
<li>
<strong>PCR Amplification:</strong> Relies on the double helix structure, strand separation (denaturation by heat), primer annealing (based on complementarity and hydrogen bonding), and enzymatic synthesis (following the 5’-3’ directionality)</li>
<li>
<strong>Hybridization Techniques (FISH, Southern/Northern blots, Microarrays):</strong> Depend on the denaturation of double strands and the specific re-annealing (hybridization) of labeled probes to complementary target sequences based on A-T/U and G-C pairing</li>
<li>
<strong>Sequencing:</strong> Determines the precise order of bases along the backbone, revealing the genetic code. Techniques often rely on enzymatic synthesis (Sanger) or detecting base incorporation during synthesis (NGS)</li>
<li>
<strong>Electrophoresis:</strong> The uniform negative charge of the sugar-phosphate backbone allows DNA and RNA fragments to migrate through a gel matrix towards the positive electrode, separating primarily based on size</li>
<li>
<strong>Restriction Enzyme Digestion:</strong> These enzymes recognize specific short DNA sequences (determined by the bases) and cut the phosphodiester backbone at precise locations</li>
<li>
<strong>Drug Design:</strong> Many drugs target nucleic acid structure or enzymes that interact with them (e.g., intercalating agents that slip between base pairs, or drugs that inhibit polymerases)</li>
</ul>
</div>
<div id="key-terms-2" class="section level2 unnumbered">
<h2>
<strong>Key Terms</strong><a class="anchor" aria-label="anchor" href="#key-terms-2"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>Nucleotide:</strong> The monomer unit of nucleic acids, consisting of a nitrogenous base, a pentose sugar (deoxyribose or ribose), and one or more phosphate groups</li>
<li>
<strong>Phosphodiester Bond:</strong> The covalent bond linking adjacent nucleotides in a nucleic acid strand. It connects the 5’ phosphate group of one nucleotide to the 3’ hydroxyl group of the previous nucleotide’s sugar</li>
<li>
<strong>Sugar-Phosphate Backbone:</strong> The repeating chain of alternating sugar and phosphate units formed by phosphodiester bonds in a nucleic acid strand. The bases project from this backbone</li>
<li>
<strong>N-Glycosidic Bond:</strong> The covalent bond linking the nitrogenous base to the 1’ carbon of the pentose sugar</li>
<li>
<strong>Directionality (5’ to 3’):</strong> The intrinsic orientation of a nucleic acid strand, defined by the phosphodiester linkages, resulting in distinct 5’ (phosphate) and 3’ (hydroxyl) ends. Sequences are read 5’ to 3’</li>
<li>
<strong>5’ End:</strong> The end of a nucleic acid strand terminating with a phosphate group attached to the 5’ carbon of the terminal sugar</li>
<li>
<strong>3’ End:</strong> The end of a nucleic acid strand terminating with a hydroxyl group attached to the 3’ carbon of the terminal sugar. Site of nucleotide addition during synthesis</li>
<li>
<strong>Double Helix:</strong> The typical secondary structure of DNA, consisting of two antiparallel polynucleotide strands wound around a central axis, held together by complementary base pairing</li>
<li>
<strong>Antiparallel:</strong> The orientation of the two strands in a DNA double helix, where one strand runs 5’ to 3’ and the complementary strand runs 3’ to 5’</li>
<li>
<strong>Complementary Base Pairing:</strong> Specific hydrogen bonding between A and T (or U) and between G and C in nucleic acids</li>
<li>
<strong>Hydrogen Bond:</strong> Weak electrostatic attraction holding complementary base pairs together. Two bonds between A-T/U, three bonds between G-C</li>
<li>
<strong>Major Groove / Minor Groove:</strong> The two distinct grooves winding along the surface of the DNA double helix, formed by the arrangement of the sugar-phosphate backbones. Sites for protein-DNA interactions</li>
<li>
<strong>Secondary Structure (RNA):</strong> Local folding patterns of an RNA molecule, such as hairpins, loops, and bulges, formed by intramolecular base pairing</li>
<li>
<strong>Tertiary Structure (RNA):</strong> The overall three-dimensional shape of an RNA molecule, resulting from interactions between secondary structural elements</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="bases.html">Bases</a></div>
<div class="next"><a href="associated-proteins.html">Associated Proteins</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#chemical-structure">Chemical Structure</a></li>
<li><a class="nav-link" href="#the-monomer-nucleotides">The Monomer: Nucleotides</a></li>
<li><a class="nav-link" href="#the-backbone-linking-nucleotides-with-phosphodiester-bonds">The Backbone: Linking Nucleotides with Phosphodiester Bonds</a></li>
<li><a class="nav-link" href="#directionality-the-5-and-3-ends">Directionality: The 5’ and 3’ Ends</a></li>
<li><a class="nav-link" href="#dna-structure-the-double-helix">DNA Structure: The Double Helix</a></li>
<li><a class="nav-link" href="#rna-structure-more-versatile">RNA Structure: More Versatile</a></li>
<li><a class="nav-link" href="#clinical-laboratory-relevance-2">Clinical Laboratory Relevance</a></li>
<li><a class="nav-link" href="#key-terms-2">Key Terms</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/01-A-03.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/01-A-03.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
