<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Gel | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="Gel Electrophoresis, using either agarose or polyacrylamide, is a cornerstone technique in the clinical molecular lab. It allows us to separate DNA (and RNA) fragments based on size, providing...">
<meta name="generator" content="bookdown 0.45 with bs4_book()">
<meta property="og:title" content="Gel | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="Gel Electrophoresis, using either agarose or polyacrylamide, is a cornerstone technique in the clinical molecular lab. It allows us to separate DNA (and RNA) fragments based on size, providing...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Gel | MB">
<meta name="twitter:description" content="Gel Electrophoresis, using either agarose or polyacrylamide, is a cornerstone technique in the clinical molecular lab. It allows us to separate DNA (and RNA) fragments based on size, providing...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-detection.html">Separation &amp; Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="active" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="" href="array-technology-1.html">Array Technology</a></li>
<li><a class="" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
<li class="book-part">Laboratory Operations</li>
<li><a class="" href="contamination.html">Contamination</a></li>
<li><a class="" href="prevention-1.html">Prevention</a></li>
<li><a class="" href="monitoring-detection-1.html">Monitoring &amp; Detection</a></li>
<li><a class="" href="elimination-1.html">Elimination</a></li>
<li><a class="" href="quality-assurance.html">Quality Assurance</a></li>
<li><a class="" href="specimen-handling.html">Specimen Handling</a></li>
<li><a class="" href="specimen-evaluation.html">Specimen Evaluation</a></li>
<li><a class="" href="nucleic-acid-evaluation.html">Nucleic Acid Evaluation</a></li>
<li><a class="" href="reagents.html">Reagents</a></li>
<li><a class="" href="assays.html">Assays</a></li>
<li><a class="" href="assay-troubleshooting-1.html">Assay Troubleshooting</a></li>
<li><a class="" href="result-calculation-interpretation-reporting-1.html">Result Calculation, Interpretation, &amp; Reporting</a></li>
<li><a class="" href="qc-pt.html">QC &amp; PT</a></li>
<li><a class="" href="assay-controls.html">Assay Controls</a></li>
<li><a class="" href="proficiency-testing.html">Proficiency Testing</a></li>
<li><a class="" href="equipment-instrumentation-1.html">Equipment &amp; Instrumentation</a></li>
<li><a class="" href="guidelines-regulations.html">Guidelines &amp; Regulations</a></li>
<li><a class="" href="test-system-categories-1.html">Test System Categories</a></li>
<li><a class="" href="regulations-standards-1.html">Regulations &amp; Standards</a></li>
<li><a class="" href="personnel.html">Personnel</a></li>
<li><a class="" href="continuing-education-1.html">Continuing Education</a></li>
<li><a class="" href="competency.html">Competency</a></li>
<li><a class="" href="safety.html">Safety</a></li>
<li><a class="" href="hazardous-materials.html">Hazardous Materials</a></li>
<li><a class="" href="biological.html">Biological</a></li>
<li><a class="" href="chemical.html">Chemical</a></li>
<li><a class="" href="laboratory-administration.html">Laboratory Administration</a></li>
<li><a class="" href="financial-management-1.html">Financial Management</a></li>
<li><a class="" href="budgets-1.html">Budgets</a></li>
<li><a class="" href="capital-equipment.html">Capital Equipment</a></li>
<li><a class="" href="cost-analysis-reimbursement-1.html">Cost Analysis &amp; Reimbursement</a></li>
<li><a class="" href="purchasing-inventory-1.html">Purchasing &amp; Inventory</a></li>
<li><a class="" href="operations-management-1.html">Operations Management</a></li>
<li><a class="" href="laboratory-information-system.html">Laboratory Information System</a></li>
<li><a class="" href="facilities-management-1.html">Facilities Management</a></li>
<li><a class="" href="intrainterdepartmental-relations-1.html">Intra/Interdepartmental Relations</a></li>
<li><a class="" href="personnel-management-1.html">Personnel Management</a></li>
<li><a class="" href="motivation-1.html">Motivation</a></li>
<li><a class="" href="staffing-productivity-1.html">Staffing &amp; Productivity</a></li>
<li><a class="" href="counseling-disciplinary-action-1.html">Counseling &amp; Disciplinary Action</a></li>
<li><a class="" href="quality-management-1.html">Quality Management</a></li>
<li><a class="" href="advanced-statistical-analysis-1.html">Advanced Statistical Analysis</a></li>
<li><a class="" href="assay-method-instrument-selection.html">Assay, Method, &amp; Instrument Selection</a></li>
<li><a class="" href="assay-validation-verification.html">Assay Validation &amp; Verification</a></li>
<li><a class="" href="quantitative-calculations-1.html">Quantitative Calculations</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="gel" class="section level1 unnumbered">
<h1>Gel<a class="anchor" aria-label="anchor" href="#gel"><i class="fas fa-link"></i></a>
</h1>
<p><strong>Gel Electrophoresis</strong>, using either agarose or polyacrylamide, is a cornerstone technique in the clinical molecular lab. It allows us to separate DNA (and RNA) fragments based on size, providing critical information for verifying PCR results, analyzing genetic variations (like RFLPs), assessing nucleic acid quality, and serving as a component of many other molecular workflows</p>
<p>Gel electrophoresis technique is fundamental for visualizing and analyzing nucleic acids (DNA and RNA). Think of it as a molecular race track or sieve where we separate molecules based primarily on their <strong>size</strong></p>
<div id="gel-electrophoresis-separating-molecules-by-size" class="section level2 unnumbered">
<h2>
<strong>Gel Electrophoresis: Separating Molecules by Size</strong><a class="anchor" aria-label="anchor" href="#gel-electrophoresis-separating-molecules-by-size"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>The Core Principle:</strong> It’s beautifully simple!
<ol style="list-style-type: decimal">
<li>
<strong>Charge:</strong> Nucleic acids (DNA/RNA) have a negatively charged phosphate backbone</li>
<li>
<strong>Electric Field:</strong> When placed in an electric field (generated by a power supply connected to an electrophoresis chamber), these negatively charged molecules will migrate towards the positive electrode (anode, usually red)</li>
<li>
<strong>The Matrix (The Gel):</strong> This is where the separation happens! The gel acts as a porous matrix, a bit like a microscopic sieve or obstacle course</li>
<li>
<strong>Size Matters:</strong> Smaller molecules navigate this matrix more easily and quickly, traveling further down the gel in a given amount of time. Larger molecules get tangled up more, move slower, and therefore travel shorter distances. So, <strong>smaller fragments migrate further, larger fragments stay closer to the starting wells.</strong>
</li>
</ol>
</li>
</ul>
<p>We primarily use two types of gels in the clinical molecular lab for nucleic acid separation: <strong>Agarose</strong> and <strong>Polyacrylamide</strong></p>
</div>
<div id="agarose-gel-electrophoresis" class="section level2 unnumbered">
<h2>
<strong>Agarose Gel Electrophoresis</strong><a class="anchor" aria-label="anchor" href="#agarose-gel-electrophoresis"><i class="fas fa-link"></i></a>
</h2>
<p>This is probably the most common type you’ll encounter for routine DNA analysis</p>
<ul>
<li>
<strong>What it is:</strong> Agarose is a natural polysaccharide extracted from seaweed. It’s purchased as a powder</li>
<li>
<strong>How it’s made</strong>
<ol style="list-style-type: decimal">
<li>Mix agarose powder with a buffer solution (like TAE - Tris-acetate-EDTA, or TBE - Tris-borate-EDTA). The buffer provides ions for conductivity and maintains pH</li>
<li>Heat the mixture (e.g., in a microwave) until the agarose dissolves completely (solution becomes clear)</li>
<li>Cool it slightly (so you don’t melt the plastic tray!)</li>
<li>Pour the molten agarose into a casting tray containing a “comb”. The comb creates the wells where you’ll load your samples</li>
<li>Let it solidify (it forms a semi-solid, Jell-O like matrix with pores)</li>
<li>Place the solidified gel (often in its tray) into the electrophoresis chamber (“gel box”), cover it with buffer, and carefully remove the comb</li>
</ol>
</li>
<li>
<strong>Pore Size &amp; Resolution</strong>
<ul>
<li>Forms relatively large pores compared to polyacrylamide</li>
<li>The pore size can be adjusted by changing the <strong>concentration:</strong> of agarose (e.g., 0.8% to 2.5%)
<ul>
<li>
<strong>Lower % Agarose:</strong> Larger pores, better for separating <em>very large</em> DNA fragments (e.g., &gt; 5-10 kilobases, kb). Gel is more fragile</li>
<li>
<strong>Higher % Agarose:</strong> Smaller pores, better for separating <em>smaller</em> DNA fragments (e.g., 50 base pairs, bp, to 2 kb). Gel is firmer</li>
</ul>
</li>
<li>Generally used to resolve DNA fragments that differ in size by at least 50-100 bp, depending on the fragment size and gel percentage</li>
</ul>
</li>
<li>
<strong>Typical Format:</strong> Horizontal “submarine” gels, where the gel lies flat in the buffer tank</li>
<li>
<strong>Clinical Applications</strong>
<ul>
<li>
<strong>Confirming PCR products:</strong> Checking if your PCR worked and if the amplified product is the expected size. This is super common!</li>
<li>
<strong>Restriction Fragment Length Polymorphism (RFLP) analysis:</strong> Analyzing fragment sizes after cutting DNA with restriction enzymes (a more traditional technique)</li>
<li><strong>Checking plasmid integrity.</strong></li>
<li>Quick check of genomic DNA quality/presence (though high MW DNA doesn’t resolve well into distinct bands)</li>
</ul>
</li>
<li>
<strong>Pros:</strong> Easy and quick to prepare, relatively non-toxic (the agarose itself), good for a wide range of <em>larger</em> DNA sizes</li>
<li>
<strong>Cons:</strong> Lower resolving power than polyacrylamide (can’t easily separate very small fragments or those differing by only a few base pairs), can be fragile at low concentrations</li>
</ul>
</div>
<div id="polyacrylamide-gel-electrophoresis-page" class="section level2 unnumbered">
<h2>
<strong>Polyacrylamide Gel Electrophoresis (PAGE)</strong><a class="anchor" aria-label="anchor" href="#polyacrylamide-gel-electrophoresis-page"><i class="fas fa-link"></i></a>
</h2>
<p>When you need higher resolution, especially for smaller fragments or even single-base differences, PAGE is the way to go</p>
<ul>
<li>
<strong>What it is:</strong> A synthetic gel matrix created by polymerizing acrylamide monomers with a cross-linking agent (usually N,N’-methylenebisacrylamide or “bis-acrylamide”)</li>
<li>
<strong>How it’s made</strong>
<ol style="list-style-type: decimal">
<li>Mix acrylamide and bis-acrylamide monomers, buffer, and water</li>
<li>Initiate polymerization chemically using catalysts like <strong>APS (ammonium persulfate):</strong> and <strong>TEMED (tetramethylethylenediamine)</strong>. <em>Caution: Unpolymerized acrylamide is a neurotoxin! Handle with care, gloves mandatory</em>
</li>
<li>The solution is quickly poured between two glass plates separated by spacers, with a comb inserted at the top to form wells</li>
<li>Polymerization occurs, forming a thin, firm gel slab</li>
</ol>
</li>
<li>
<strong>Pore Size &amp; Resolution</strong>
<ul>
<li>Forms much smaller, more uniform pores than agarose</li>
<li>Pore size is precisely controlled by:
<ul>
<li>
<strong>Total acrylamide concentration (%T):</strong> Higher concentration = smaller pores</li>
<li>
<strong>Amount of cross-linker (%C):</strong> Affects pore structure</li>
</ul>
</li>
<li>Provides <strong>very high resolution</strong>, capable of separating molecules that differ by only a single base pair (essential for older Sanger sequencing methods) or distinguishing small DNA/RNA fragments very clearly</li>
<li>Excellent for fragments from ~5 bp up to ~1000 bp (1 kb)</li>
</ul>
</li>
<li>
<strong>Typical Format:</strong> Vertical gels, where the thin gel slab stands upright between the buffer chambers</li>
<li>
<strong>Clinical Applications</strong>
<ul>
<li>
<strong>Analysis of small PCR products:</strong> requiring high resolution</li>
<li>
<strong>Mutation detection techniques:</strong> like Single-Strand Conformation Polymorphism (SSCP) or heteroduplex analysis (less common now but rely on PAGE)</li>
<li>Historically crucial for <strong>Sanger sequencing:</strong> fragment separation</li>
<li>Analysis of short tandem repeats (STRs) - though capillary electrophoresis is now standard</li>
<li><em>Also the standard for protein separation (SDS-PAGE), but our focus here is nucleic acids</em></li>
</ul>
</li>
<li>
<strong>Pros:</strong> Superior resolution for small fragments, gels are mechanically stable</li>
<li>
<strong>Cons:</strong> Acrylamide monomer is toxic before polymerization, gels are more complex and time-consuming to prepare than agarose</li>
</ul>
</div>
<div id="comparison-agarose-vs.-polyacrylamide" class="section level2 unnumbered">
<h2>
<strong>Comparison: Agarose vs. Polyacrylamide</strong><a class="anchor" aria-label="anchor" href="#comparison-agarose-vs.-polyacrylamide"><i class="fas fa-link"></i></a>
</h2>
<div class="inline-table"><table class="table table-sm">
<colgroup>
<col width="20%">
<col width="39%">
<col width="40%">
</colgroup>
<thead><tr class="header">
<th align="left">Feature</th>
<th align="left">Agarose Gel</th>
<th align="left">Polyacrylamide Gel (PAGE)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left"><strong>Material</strong></td>
<td align="left">Natural Polysaccharide (Seaweed)</td>
<td align="left">Synthetic Polymer (Acrylamide)</td>
</tr>
<tr class="even">
<td align="left"><strong>Pore Size</strong></td>
<td align="left">Larger, Less Uniform</td>
<td align="left">Smaller, More Uniform &amp; Controllable</td>
</tr>
<tr class="odd">
<td align="left"><strong>Resolution</strong></td>
<td align="left">Lower (Good for &gt;50 bp difference)</td>
<td align="left">High (Can resolve 1 bp difference)</td>
</tr>
<tr class="even">
<td align="left"><strong>Typical Range</strong></td>
<td align="left">~50 bp to &gt;20 kb</td>
<td align="left">~5 bp to ~1 kb</td>
</tr>
<tr class="odd">
<td align="left"><strong>Typical Format</strong></td>
<td align="left">Horizontal (“Submarine”)</td>
<td align="left">Vertical</td>
</tr>
<tr class="even">
<td align="left"><strong>Primary NA Use</strong></td>
<td align="left">PCR products, RFLP, Plasmids</td>
<td align="left">Sequencing (classic), Small fragments, Mutation analysis</td>
</tr>
<tr class="odd">
<td align="left"><strong>Toxicity</strong></td>
<td align="left">Gel itself is non-toxic</td>
<td align="left">Monomer is a neurotoxin</td>
</tr>
<tr class="even">
<td align="left"><strong>Preparation</strong></td>
<td align="left">Simple (dissolve, pour, cool)</td>
<td align="left">More complex (polymerization rxn)</td>
</tr>
</tbody>
</table></div>
</div>
<div id="essential-components-the-process" class="section level2 unnumbered">
<h2>
<strong>Essential Components &amp; The Process</strong><a class="anchor" aria-label="anchor" href="#essential-components-the-process"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li><p><strong>Gel:</strong> Agarose or Polyacrylamide matrix</p></li>
<li><p><strong>Electrophoresis Buffer (TAE or TBE):</strong> Fills the tank, covers the gel, provides ions for current, maintains pH</p></li>
<li><p><strong>Power Supply:</strong> Provides the electric current (controlled voltage)</p></li>
<li><p><strong>Gel Box/Apparatus:</strong> Holds the gel and buffer, has electrodes (+ and -)</p></li>
<li>
<p><strong>Samples:</strong> Your DNA/RNA! Usually mixed with <strong>Loading Dye</strong></p>
<ul>
<li>
<strong>Loading Dye:</strong> Contains glycerol or sucrose (makes sample dense so it sinks into the well) and one or more tracking dyes (e.g., bromophenol blue, xylene cyanol) that migrate at predictable rates, allowing you to monitor the run’s progress. <em>Loading dye does NOT stain the DNA itself</em>
</li>
</ul>
</li>
<li><p><strong>DNA Size Standard (“Ladder”):</strong> A mixture of DNA fragments of known sizes, run in one lane for comparison to estimate the size of your sample fragments</p></li>
<li><p><strong>Detection/Visualization System:</strong> How you see the results after the run</p></li>
<li>
<p><strong>Basic Steps</strong></p>
<ol style="list-style-type: decimal">
<li>Prepare the gel</li>
<li>Place gel in the buffer-filled tank</li>
<li>Mix samples with loading dye</li>
<li>Carefully load samples and DNA ladder into the wells</li>
<li>Connect the power supply (ensure correct orientation: wells near negative electrode!)</li>
<li>Run the gel at a set voltage/current for a specific time</li>
<li>Stop the run before the tracking dye runs off the end</li>
<li>Stain the gel (if stain wasn’t included during casting)</li>
<li>Visualize the separated fragments (bands) using an appropriate imaging system (e.g., UV light for EtBr/SYBR)</li>
</ol>
</li>
</ul>
</div>
<div id="detection-seeing-the-bands" class="section level2 unnumbered">
<h2>
<strong>Detection: Seeing the Bands</strong><a class="anchor" aria-label="anchor" href="#detection-seeing-the-bands"><i class="fas fa-link"></i></a>
</h2>
<p>Since nucleic acids are invisible to the naked eye, we need to stain them. Common methods:</p>
<ul>
<li>
<strong>Intercalating Dyes:</strong> These molecules slip in between the stacked base pairs of the DNA double helix. When exposed to specific wavelengths of light (usually UV or blue light), they fluoresce, revealing the location of the DNA bands
<ul>
<li>
<strong>Ethidium Bromide (EtBr):</strong> The classic. Very sensitive, fluoresces orange under UV. <em>BUT it’s a potent mutagen - handle with extreme caution and dispose of properly!</em>
</li>
<li>
<strong>SYBR Safe, SYBR Gold, GelRed, GelGreen:</strong> Newer, much safer alternatives designed to be less mutagenic. Often fluoresce green or orange under blue or UV light. They are often preferred in clinical labs now</li>
</ul>
</li>
<li>
<strong>Autoradiography:</strong> Used if the nucleic acids were labeled with radioactivity (e.g., ³²P). The gel is exposed to X-ray film, and the radioactive bands create dark spots on the developed film. (Less common now due to safety and disposal issues)</li>
</ul>
</div>
<div id="key-factors-influencing-migration-rate" class="section level2 unnumbered">
<h2>
<strong>Key Factors Influencing Migration Rate</strong><a class="anchor" aria-label="anchor" href="#key-factors-influencing-migration-rate"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>Molecular Size:</strong> The <em>primary</em> determinant. Smaller = faster</li>
<li>
<strong>Agarose/Acrylamide Concentration:</strong> Higher concentration = smaller pores = slower migration, especially for larger fragments</li>
<li>
<strong>Voltage:</strong> Higher voltage = faster migration. But too high generates heat, which can melt agarose gels or cause band distortion (“smiling”)</li>
<li>
<strong>Conformation of DNA:</strong> Supercoiled plasmids, nicked circles, and linear DNA of the <em>same mass</em> migrate differently (supercoiled is fastest in agarose). Less critical for PCR products which are linear</li>
<li>
<strong>Buffer Composition &amp; Ionic Strength:</strong> Affects conductivity and pH. Buffer exhaustion during long runs can affect migration</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="electrophoresis.html">Electrophoresis</a></div>
<div class="next"><a href="capillary.html">Capillary</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#gel">Gel</a></li>
<li><a class="nav-link" href="#gel-electrophoresis-separating-molecules-by-size">Gel Electrophoresis: Separating Molecules by Size</a></li>
<li><a class="nav-link" href="#agarose-gel-electrophoresis">Agarose Gel Electrophoresis</a></li>
<li><a class="nav-link" href="#polyacrylamide-gel-electrophoresis-page">Polyacrylamide Gel Electrophoresis (PAGE)</a></li>
<li><a class="nav-link" href="#comparison-agarose-vs.-polyacrylamide">Comparison: Agarose vs. Polyacrylamide</a></li>
<li><a class="nav-link" href="#essential-components-the-process">Essential Components &amp; The Process</a></li>
<li><a class="nav-link" href="#detection-seeing-the-bands">Detection: Seeing the Bands</a></li>
<li><a class="nav-link" href="#key-factors-influencing-migration-rate">Key Factors Influencing Migration Rate</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/02-B-01-a.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/02-B-01-a.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
