<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Multiplex Ligation-Dependent Probe Amplification (MLPA) | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="MLPA is a highly sensitive and elegant technique used in the clinical lab to rapidly determine the copy number of specific DNA sequences. This is especially useful for detecting copy number...">
<meta name="generator" content="bookdown 0.43 with bs4_book()">
<meta property="og:title" content="Multiplex Ligation-Dependent Probe Amplification (MLPA) | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="MLPA is a highly sensitive and elegant technique used in the clinical lab to rapidly determine the copy number of specific DNA sequences. This is especially useful for detecting copy number...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Multiplex Ligation-Dependent Probe Amplification (MLPA) | MB">
<meta name="twitter:description" content="MLPA is a highly sensitive and elegant technique used in the clinical lab to rapidly determine the copy number of specific DNA sequences. This is especially useful for detecting copy number...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-and-detection.html">Separation and Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="" href="array-technology-1.html">Array Technology</a></li>
<li><a class="active" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="multiplex-ligation-dependent-probe-amplification-mlpa-1" class="section level1 unnumbered">
<h1>Multiplex Ligation-Dependent Probe Amplification (MLPA)<a class="anchor" aria-label="anchor" href="#multiplex-ligation-dependent-probe-amplification-mlpa-1"><i class="fas fa-link"></i></a>
</h1>
<p><strong>MLPA</strong> is a highly sensitive and elegant technique used in the clinical lab to rapidly determine the copy number of specific DNA sequences. This is especially useful for detecting <strong>copy number variations (CNVs)</strong>, which are gains (duplications) or losses (deletions) of DNA regions, or for looking at the methylation status of a large number of targets simultaneously</p>
<p>Think of MLPA as a specialized, sophisticated version of PCR. It allows you to analyze <strong>50 to 60 targets in a single reaction</strong>, making it incredibly efficient for diagnostics. It relies on three crucial components working in concert: ligation, amplification, and size separation</p>
<div id="the-core-principle-counting-dna-via-ligation-and-pcr" class="section level2 unnumbered">
<h2>
<strong>The Core Principle: Counting DNA via Ligation and PCR</strong><a class="anchor" aria-label="anchor" href="#the-core-principle-counting-dna-via-ligation-and-pcr"><i class="fas fa-link"></i></a>
</h2>
<p>MLPA’s ingenious design allows it to measure the <strong>quantity</strong> of target DNA, rather than just its presence or absence. The key is to link two separate probe halves together only when both are perfectly hybridized to the target DNA. This linked sequence can then be amplified by PCR</p>
<p><strong>Here is the MLPA magic in steps</strong></p>
<div id="probe-hybridization-1" class="section level3 unnumbered">
<h3>
<strong>1. Probe Hybridization</strong><a class="anchor" aria-label="anchor" href="#probe-hybridization-1"><i class="fas fa-link"></i></a>
</h3>
<p>The assay uses sets of special MLPA probes. Each probe consists of two separate, single-stranded oligonucleotides (a Left Probe Oligo and a Right Probe Oligo) that are designed to bind adjacently to a specific target sequence in the patient’s DNA (which is added in excess)</p>
<ul>
<li>
<strong>Left Probe Oligo (LPO):</strong> Contains a short sequence that binds to the target DNA. At its 3’ end, it has a universal PCR primer site</li>
<li>
<strong>Right Probe Oligo (RPO):</strong> Contains a short sequence that binds to the target DNA immediately adjacent to the LPO. At its 5’ end, it has a universal PCR primer site</li>
<li>
<strong>Stuffer Sequence (unique to each probe pair):</strong> This is a non-specific DNA segment located on either the LPO or RPO. Each pair of probes targeting a different gene has a <em>different length</em> stuffer sequence</li>
</ul>
</div>
<div id="ligation" class="section level3 unnumbered">
<h3>
<strong>2. Ligation</strong><a class="anchor" aria-label="anchor" href="#ligation"><i class="fas fa-link"></i></a>
</h3>
<p>A <strong>ligase enzyme</strong> (the molecular glue) is added. Ligase can only join the LPO and RPO together if they are perfectly hybridized adjacent to each other on the patient’s template DNA. Crucially, even a single-base mismatch prevents the ligation from occurring</p>
<ul>
<li>If the target sequence is present and complementary, the probes ligate into a single, contiguous molecule</li>
</ul>
</div>
<div id="amplification-pcr" class="section level3 unnumbered">
<h3>
<strong>3. Amplification (PCR)</strong><a class="anchor" aria-label="anchor" href="#amplification-pcr"><i class="fas fa-link"></i></a>
</h3>
<p>Once the probes are ligated, they can be amplified using a <strong>single, universal pair of PCR primers</strong> that bind to the universal primer sites (added in step 1) on the 5’ end of the LPO and the 3’ end of the RPO</p>
<ul>
<li>All the different MLPA probes (even those targeting different genes) can be amplified by this one universal primer pair. This makes it a <strong>multiplex PCR</strong>
</li>
</ul>
</div>
<div id="separation-and-quantification" class="section level3 unnumbered">
<h3>
<strong>4. Separation and Quantification</strong><a class="anchor" aria-label="anchor" href="#separation-and-quantification"><i class="fas fa-link"></i></a>
</h3>
<p>The PCR products are then separated by size using <strong>capillary electrophoresis (CE)</strong>. Because each probe set has a unique-length stuffer sequence, the different ligated products (each representing a different gene target) will separate into distinct peaks</p>
<ul>
<li>The crucial insight: The <strong>peak area</strong> for each resulting PCR product is directly proportional to the amount of <em>starting</em> target DNA in the sample</li>
</ul>
</div>
</div>
<div id="interpreting-mlpa-results-the-copy-number" class="section level2 unnumbered">
<h2>
<strong>Interpreting MLPA Results: The Copy Number</strong><a class="anchor" aria-label="anchor" href="#interpreting-mlpa-results-the-copy-number"><i class="fas fa-link"></i></a>
</h2>
<p>By analyzing the peak heights (or areas) on the electropherogram, we can determine the copy number of the target sequence</p>
<ul>
<li>
<strong>Normalization:</strong> The peak areas from the patient sample are compared to a normal reference sample to account for minor variations in the reaction. We also include probes for sequences expected to be present in normal copy number</li>
<li>
<strong>Copy Number Calculation</strong>
<ul>
<li>
<strong>Normal Sample:</strong> For an autosomal gene, we expect a copy number of 2</li>
<li>
<strong>Deletion (Loss):</strong> If a target sequence has a deletion on one chromosome, the amount of template is halved, and the corresponding PCR peak will be about half the size of the normal controls. Copy number = 1</li>
<li>
<strong>Duplication (Gain):</strong> If a target sequence has a duplication, the corresponding peak will be about 1.5 times the size of the normal controls. Copy number = 3</li>
</ul>
</li>
</ul>
</div>
<div id="clinical-applications-in-the-molecular-lab-2" class="section level2 unnumbered">
<h2>
<strong>Clinical Applications in the Molecular Lab</strong><a class="anchor" aria-label="anchor" href="#clinical-applications-in-the-molecular-lab-2"><i class="fas fa-link"></i></a>
</h2>
<p>MLPA is a powerful tool for analyzing targeted gene sequences and detecting CNVs that are too small to be seen by conventional karyotyping but too large to be efficiently picked up by high-throughput sequencing</p>
<ul>
<li>
<strong>Detection of Gene Deletions and Duplications:</strong> This is the primary use of MLPA
<ul>
<li>
<strong>Examples:</strong> Identifying deletions or duplications in genes like <em>DMD</em> (Duchenne Muscular Dystrophy), <em>BRCA1/BRCA2</em> (hereditary breast/ovarian cancer), or <strong>MSH2</strong> in Lynch syndrome</li>
<li>
<strong>Syndrome Diagnosis:</strong> MLPA is frequently used for detecting microdeletion syndromes like <strong>DiGeorge syndrome</strong> or <strong>Prader-Willi/Angelman syndrome</strong> (for the latter, using specialized probes that detect methylation status)</li>
</ul>
</li>
<li>
<strong>Tumor-Specific Deletions/Amplifications:</strong> Identifying chromosomal gains or losses associated with different types of cancers</li>
<li>
<strong>Methylation Analysis:</strong> Special MLPA probes (Methylation-sensitive MLPA or MS-MLPA) can detect epigenetic changes. These probes contain a restriction site that is only cut by a methylation-sensitive enzyme if the DNA is unmethylated. The presence or absence of a peak then indicates the methylation status</li>
</ul>
</div>
<div id="advantages-vs.-limitations-1" class="section level2 unnumbered">
<h2>
<strong>Advantages vs. Limitations</strong><a class="anchor" aria-label="anchor" href="#advantages-vs.-limitations-1"><i class="fas fa-link"></i></a>
</h2>
<div id="advantages-2" class="section level3 unnumbered">
<h3>
<strong>Advantages</strong><a class="anchor" aria-label="anchor" href="#advantages-2"><i class="fas fa-link"></i></a>
</h3>
<ul>
<li>
<strong>High Throughput and Multiplexing:</strong> Detects many targets in a single reaction (up to 60)</li>
<li>
<strong>Quantitative:</strong> Provides precise copy number data (1, 2, 3, etc.)</li>
<li>
<strong>Cost-Effective:</strong> Much cheaper than whole-exome or whole-genome sequencing for analyzing specific sets of genes</li>
<li>
<strong>High Specificity:</strong> Requires the ligation event for amplification to occur, making it very sensitive and specific. Even a single nucleotide difference prevents ligation</li>
</ul>
</div>
<div id="limitations-2" class="section level3 unnumbered">
<h3>
<strong>Limitations</strong><a class="anchor" aria-label="anchor" href="#limitations-2"><i class="fas fa-link"></i></a>
</h3>
<ul>
<li>
<strong>Limited Coverage:</strong> It only provides information on the genes that have probes included in the mix. It cannot detect variants in genes not targeted by the probes</li>
<li>
<strong>Cannot Detect SNVs:</strong> It detects CNVs (deletions/duplications) but generally cannot detect point mutations (SNVs) or indels unless the mutation falls directly at the probe ligation site</li>
<li>
<strong>Requires Specific Probe Design:</strong> Designing effective MLPA probe sets requires careful selection and validation</li>
<li>
<strong>Artifacts:</strong> Can be sensitive to differences in sample quality, temperature variations, and impurities</li>
</ul>
</div>
</div>
<div id="key-terms-42" class="section level2 unnumbered">
<h2>
<strong>Key Terms</strong><a class="anchor" aria-label="anchor" href="#key-terms-42"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li>
<strong>MLPA (Multiplex Ligation-Dependent Probe Amplification):</strong> A molecular technique that allows for the simultaneous detection and quantification of up to 60 different DNA sequences in a single reaction</li>
<li>
<strong>Copy Number Variation (CNV):</strong> A segment of DNA that is 1 kilobase (kb) or larger and is present in variable copy numbers in different individuals. MLPA is a key method for detecting CNVs (deletions or duplications)</li>
<li>
<strong>Probe (MLPA):</strong> A pair of two short oligonucleotides (Left and Right Probe Oligos) designed to bind adjacently to a specific target sequence</li>
<li>
<strong>Ligation:</strong> The crucial enzymatic process where DNA ligase joins the two separate MLPA probes together when they are hybridized adjacent to each other on the template DNA</li>
<li>
<strong>Stuffer Sequence:</strong> A DNA segment of unique, defined length included in each probe pair. This allows different targets to be separated and distinguished by size on capillary electrophoresis</li>
<li>
<strong>Universal Primer:</strong> A single pair of primers that can amplify all of the ligated MLPA probes, regardless of the target sequence, as long as they contain the universal primer binding sites</li>
<li>
<strong>Electropherogram (MLPA):</strong> The graphical output of the capillary electrophoresis run, showing distinct peaks for each target sequence; the height or area of the peak corresponds to the copy number of that target in the sample</li>
<li>
<strong>Methylation-Sensitive MLPA (MS-MLPA):</strong> A variant of MLPA used to detect the methylation status of DNA, typically in regions associated with specific genetic disorders or cancers, using methylation-sensitive restriction enzymes</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="array-technology-1.html">Array Technology</a></div>
<div class="next"><a href="mass-spectrometry.html">Mass Spectrometry</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#multiplex-ligation-dependent-probe-amplification-mlpa-1">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li>
<a class="nav-link" href="#the-core-principle-counting-dna-via-ligation-and-pcr">The Core Principle: Counting DNA via Ligation and PCR</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#probe-hybridization-1">1. Probe Hybridization</a></li>
<li><a class="nav-link" href="#ligation">2. Ligation</a></li>
<li><a class="nav-link" href="#amplification-pcr">3. Amplification (PCR)</a></li>
<li><a class="nav-link" href="#separation-and-quantification">4. Separation and Quantification</a></li>
</ul>
</li>
<li><a class="nav-link" href="#interpreting-mlpa-results-the-copy-number">Interpreting MLPA Results: The Copy Number</a></li>
<li><a class="nav-link" href="#clinical-applications-in-the-molecular-lab-2">Clinical Applications in the Molecular Lab</a></li>
<li>
<a class="nav-link" href="#advantages-vs.-limitations-1">Advantages vs. Limitations</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#advantages-2">Advantages</a></li>
<li><a class="nav-link" href="#limitations-2">Limitations</a></li>
</ul>
</li>
<li><a class="nav-link" href="#key-terms-42">Key Terms</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/02-E-07.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/02-E-07.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
