<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Quantitative Calculations | MB</title>
<meta name="author" content="Brett M. Rice">
<meta name="description" content="In Molecular Biology, “Quality Management” extends beyond verifying that a test is positive or negative; it involves the rigorous mathematical validation of how much target is present....">
<meta name="generator" content="bookdown 0.45 with bs4_book()">
<meta property="og:title" content="Quantitative Calculations | MB">
<meta property="og:type" content="book">
<meta property="og:description" content="In Molecular Biology, “Quality Management” extends beyond verifying that a test is positive or negative; it involves the rigorous mathematical validation of how much target is present....">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Quantitative Calculations | MB">
<meta name="twitter:description" content="In Molecular Biology, “Quality Management” extends beyond verifying that a test is positive or negative; it involves the rigorous mathematical validation of how much target is present....">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.9.0/transition.js"></script><script src="libs/bs3compat-0.9.0/tabs.js"></script><script src="libs/bs3compat-0.9.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link rel="icon" type="image/png" href="imgs/favicon.png">
<meta name="google-adsense-account" content="ca-pub-1344282930858290">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JERRQQFPF4"></script><script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-JERRQQFPF4');
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<link rel="stylesheet" href="style.css">
<link rel="stylesheet" href="toc.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Molecular Biology</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="content-areas.html">Content Areas</a></li>
<li><a class="" href="author.html">Author</a></li>
<li><a class="" href="ascp-boc.html">ASCP BOC</a></li>
<li><a class="" href="reading-list.html">Reading List</a></li>
<li class="book-part">Molecular Science</li>
<li><a class="" href="nucleic-acid-chemistry.html">Nucleic Acid Chemistry</a></li>
<li><a class="" href="sugars.html">Sugars</a></li>
<li><a class="" href="bases.html">Bases</a></li>
<li><a class="" href="chemical-structure.html">Chemical Structure</a></li>
<li><a class="" href="associated-proteins.html">Associated Proteins</a></li>
<li><a class="" href="mutations.html">Mutations</a></li>
<li><a class="" href="basic-molecular-theory.html">Basic Molecular Theory</a></li>
<li><a class="" href="replication.html">Replication</a></li>
<li><a class="" href="transcription.html">Transcription</a></li>
<li><a class="" href="exons-introns-splicing.html">Exons, Introns, &amp; Splicing</a></li>
<li><a class="" href="translation.html">Translation</a></li>
<li><a class="" href="chromosome-structure.html">Chromosome Structure</a></li>
<li><a class="" href="extrachromosomal-structure.html">Extrachromosomal Structure</a></li>
<li><a class="" href="protein-structure.html">Protein Structure</a></li>
<li><a class="" href="biochemical-reagents.html">Biochemical Reagents</a></li>
<li><a class="" href="polymerase-enzymes.html">Polymerase Enzymes</a></li>
<li><a class="" href="dna.html">DNA</a></li>
<li><a class="" href="rna.html">RNA</a></li>
<li><a class="" href="endo-exonucleases.html">Endo &amp; Exonucleases</a></li>
<li><a class="" href="reverse-rranscriptase.html">Reverse Rranscriptase</a></li>
<li><a class="" href="dna-ligase.html">DNA Ligase</a></li>
<li><a class="" href="assay-development.html">Assay Development</a></li>
<li><a class="" href="genetics.html">Genetics</a></li>
<li><a class="" href="human.html">Human</a></li>
<li><a class="" href="microbial.html">Microbial</a></li>
<li class="book-part">Molecular Techniques</li>
<li><a class="" href="nucleic-acid-isolation.html">Nucleic Acid Isolation</a></li>
<li><a class="" href="automated-methods-1.html">Automated Methods</a></li>
<li><a class="" href="manual-methods-1.html">Manual Methods</a></li>
<li><a class="" href="separation-detection.html">Separation &amp; Detection</a></li>
<li><a class="" href="electrophoresis.html">Electrophoresis</a></li>
<li><a class="" href="gel.html">Gel</a></li>
<li><a class="" href="capillary.html">Capillary</a></li>
<li><a class="" href="blotting-probing.html">Blotting &amp; Probing</a></li>
<li><a class="" href="probe-hybridization.html">Probe Hybridization</a></li>
<li><a class="" href="nucleic-acid-purification.html">Nucleic Acid Purification</a></li>
<li><a class="" href="probe-structure.html">Probe Structure</a></li>
<li><a class="" href="nucleic-acid-amplification.html">Nucleic Acid Amplification</a></li>
<li><a class="" href="pcr.html">PCR</a></li>
<li><a class="" href="oligonucleotides.html">Oligonucleotides</a></li>
<li><a class="" href="reaction-optimization.html">Reaction Optimization</a></li>
<li><a class="" href="pcr-variations.html">PCR Variations</a></li>
<li><a class="" href="other.html">Other</a></li>
<li><a class="" href="sequencing.html">Sequencing</a></li>
<li><a class="" href="sanger.html">Sanger</a></li>
<li><a class="" href="next-generation.html">Next-Generation</a></li>
<li><a class="" href="other-1.html">Other</a></li>
<li><a class="" href="bioinformatics.html">Bioinformatics</a></li>
<li><a class="" href="other-techniques.html">Other Techniques</a></li>
<li><a class="" href="melt-curve-analysis-1.html">Melt-Curve Analysis</a></li>
<li><a class="" href="nucleic-acid-labeling-1.html">Nucleic Acid Labeling</a></li>
<li><a class="" href="in-situ-hybridization-ish-1.html">In Situ Hybridization (ISH)</a></li>
<li><a class="" href="restriction-fragment-length-polymorphism-rflp-1.html">Restriction Fragment Length Polymorphism (RFLP)</a></li>
<li><a class="" href="epigenetic-modification-detection-1.html">Epigenetic Modification Detection</a></li>
<li><a class="" href="array-technology-1.html">Array Technology</a></li>
<li><a class="" href="multiplex-ligation-dependent-probe-amplification-mlpa-1.html">Multiplex Ligation-Dependent Probe Amplification (MLPA)</a></li>
<li><a class="" href="mass-spectrometry.html">Mass Spectrometry</a></li>
<li><a class="" href="multi-locus-sequence-typing-mlst-1.html">Multi-Locus Sequence Typing (MLST)</a></li>
<li class="book-part">Testing Applications</li>
<li><a class="" href="infectious-disease.html">Infectious Disease</a></li>
<li><a class="" href="qualitative-analysis.html">Qualitative Analysis</a></li>
<li><a class="" href="quantitative-analysis.html">Quantitative Analysis</a></li>
<li><a class="" href="genotypic-characterization.html">Genotypic Characterization</a></li>
<li><a class="" href="oncology.html">Oncology</a></li>
<li><a class="" href="leukemias-lymphomas-1.html">Leukemias &amp; Lymphomas</a></li>
<li><a class="" href="solid-tumors-1.html">Solid Tumors</a></li>
<li><a class="" href="hereditary-cancer-syndromes.html">Hereditary Cancer Syndromes</a></li>
<li><a class="" href="genetics-1.html">Genetics</a></li>
<li><a class="" href="hemoglobinopathies.html">Hemoglobinopathies</a></li>
<li><a class="" href="coagulopathies.html">Coagulopathies</a></li>
<li><a class="" href="trinucleotide-repeat-disorders.html">Trinucleotide Repeat Disorders</a></li>
<li><a class="" href="single-gene-disorders.html">Single Gene Disorders</a></li>
<li><a class="" href="epigenetic-disorders.html">Epigenetic Disorders</a></li>
<li><a class="" href="mitochondrial-disorders.html">Mitochondrial Disorders</a></li>
<li><a class="" href="other-2.html">Other</a></li>
<li><a class="" href="histocompatibility-1.html">Histocompatibility</a></li>
<li><a class="" href="genetic-identity-1.html">Genetic Identity</a></li>
<li><a class="" href="engraftment-1.html">Engraftment</a></li>
<li><a class="" href="pharmacogenomics.html">Pharmacogenomics</a></li>
<li class="book-part">Laboratory Operations</li>
<li><a class="" href="contamination.html">Contamination</a></li>
<li><a class="" href="prevention-1.html">Prevention</a></li>
<li><a class="" href="monitoring-detection-1.html">Monitoring &amp; Detection</a></li>
<li><a class="" href="elimination-1.html">Elimination</a></li>
<li><a class="" href="quality-assurance.html">Quality Assurance</a></li>
<li><a class="" href="specimen-handling.html">Specimen Handling</a></li>
<li><a class="" href="specimen-evaluation.html">Specimen Evaluation</a></li>
<li><a class="" href="nucleic-acid-evaluation.html">Nucleic Acid Evaluation</a></li>
<li><a class="" href="reagents.html">Reagents</a></li>
<li><a class="" href="assays.html">Assays</a></li>
<li><a class="" href="assay-troubleshooting-1.html">Assay Troubleshooting</a></li>
<li><a class="" href="result-calculation-interpretation-reporting-1.html">Result Calculation, Interpretation, &amp; Reporting</a></li>
<li><a class="" href="qc-pt.html">QC &amp; PT</a></li>
<li><a class="" href="assay-controls.html">Assay Controls</a></li>
<li><a class="" href="proficiency-testing.html">Proficiency Testing</a></li>
<li><a class="" href="equipment-instrumentation-1.html">Equipment &amp; Instrumentation</a></li>
<li><a class="" href="guidelines-regulations.html">Guidelines &amp; Regulations</a></li>
<li><a class="" href="test-system-categories-1.html">Test System Categories</a></li>
<li><a class="" href="regulations-standards-1.html">Regulations &amp; Standards</a></li>
<li><a class="" href="personnel.html">Personnel</a></li>
<li><a class="" href="continuing-education-1.html">Continuing Education</a></li>
<li><a class="" href="competency.html">Competency</a></li>
<li><a class="" href="safety.html">Safety</a></li>
<li><a class="" href="hazardous-materials.html">Hazardous Materials</a></li>
<li><a class="" href="biological.html">Biological</a></li>
<li><a class="" href="chemical.html">Chemical</a></li>
<li><a class="" href="laboratory-administration.html">Laboratory Administration</a></li>
<li><a class="" href="financial-management-1.html">Financial Management</a></li>
<li><a class="" href="budgets-1.html">Budgets</a></li>
<li><a class="" href="capital-equipment.html">Capital Equipment</a></li>
<li><a class="" href="cost-analysis-reimbursement-1.html">Cost Analysis &amp; Reimbursement</a></li>
<li><a class="" href="purchasing-inventory-1.html">Purchasing &amp; Inventory</a></li>
<li><a class="" href="operations-management-1.html">Operations Management</a></li>
<li><a class="" href="laboratory-information-system.html">Laboratory Information System</a></li>
<li><a class="" href="facilities-management-1.html">Facilities Management</a></li>
<li><a class="" href="intrainterdepartmental-relations-1.html">Intra/Interdepartmental Relations</a></li>
<li><a class="" href="personnel-management-1.html">Personnel Management</a></li>
<li><a class="" href="motivation-1.html">Motivation</a></li>
<li><a class="" href="staffing-productivity-1.html">Staffing &amp; Productivity</a></li>
<li><a class="" href="counseling-disciplinary-action-1.html">Counseling &amp; Disciplinary Action</a></li>
<li><a class="" href="quality-management-1.html">Quality Management</a></li>
<li><a class="" href="advanced-statistical-analysis-1.html">Advanced Statistical Analysis</a></li>
<li><a class="" href="assay-method-instrument-selection.html">Assay, Method, &amp; Instrument Selection</a></li>
<li><a class="" href="assay-validation-verification.html">Assay Validation &amp; Verification</a></li>
<li><a class="active" href="quantitative-calculations-1.html">Quantitative Calculations</a></li>
</ul>

        <div class="book-extra" style="display:none;">
          <p><a id="book-repo" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="quantitative-calculations-1" class="section level1 unnumbered">
<h1>Quantitative Calculations<a class="anchor" aria-label="anchor" href="#quantitative-calculations-1"><i class="fas fa-link"></i></a>
</h1>
<p>In Molecular Biology, “Quality Management” extends beyond verifying that a test is positive or negative; it involves the rigorous mathematical validation of <em>how much</em> target is present. Quantitative PCR (qPCR) is the industry standard for measuring Viral Loads (e.g., HIV, HCV, CMV) and gene expression levels (e.g., BCR-ABL fusion transcripts in leukemia). The accuracy of these patient results depends entirely on the mathematical validity of the <strong>Standard Curve</strong> generated during the run. Laboratory administration requires a deep understanding of these calculations to establish acceptance criteria and troubleshoot assay failures</p>
<div id="the-standard-curve-absolute-quantification" class="section level2 unnumbered">
<h2>The Standard Curve (Absolute Quantification)<a class="anchor" aria-label="anchor" href="#the-standard-curve-absolute-quantification"><i class="fas fa-link"></i></a>
</h2>
<p>The Standard Curve is the “ruler” used to measure the unknown patient samples. It is generated by amplifying a dilution series of a Standard with a known concentration (e.g., a calibrated plasmid or RNA transcript). The relationship between the concentration and the point at which the fluorescence crosses the background threshold (Cycle Threshold or <span class="math inline">\(C_t\)</span>) forms the basis of the calculation</p>
<ul>
<li>
<strong>Plotting the Data</strong>
<ul>
<li>The instrument plots the <strong>Logarithm of the Concentration:</strong> (copies/mL or IU/mL) on the <strong>X-axis</strong>
</li>
<li>The instrument plots the <strong><span class="math inline">\(C_t\)</span> Value:</strong> on the <strong>Y-axis</strong>
</li>
<li>Because PCR is an exponential process (doubling every cycle), plotting the Log of concentration against the linear Cycle number results in a straight line. A linear regression analysis is performed to fit a line to these points, adhering to the equation <span class="math inline">\(y = mx + b\)</span>
</li>
</ul>
</li>
<li>
<strong>The Equation Components</strong>
<ul>
<li>
<strong><span class="math inline">\(y\)</span> (<span class="math inline">\(C_t\)</span>):</strong> The cycle threshold measured by the instrument</li>
<li>
<strong><span class="math inline">\(x\)</span> (Log Concentration):</strong> The known quantity of the standard</li>
<li>
<strong><span class="math inline">\(m\)</span> (Slope):</strong> The angle of the line. This represents the <strong>Efficiency</strong> of the reaction. Steeper or flatter slopes indicate biochemical problems</li>
<li>
<strong><span class="math inline">\(b\)</span> (Y-intercept):</strong> The theoretical <span class="math inline">\(C_t\)</span> value if the concentration were 1 copy. This represents the <strong>Sensitivity</strong> of the assay</li>
</ul>
</li>
</ul>
</div>
<div id="pcr-efficiency-calculations" class="section level2 unnumbered">
<h2>PCR Efficiency Calculations<a class="anchor" aria-label="anchor" href="#pcr-efficiency-calculations"><i class="fas fa-link"></i></a>
</h2>
<p>The most critical Quality Management metric derived from the standard curve is the <strong>PCR Efficiency (<span class="math inline">\(E\)</span>)</strong>. In an ideal reaction, the amount of DNA doubles exactly every cycle (100% efficiency). If the reaction is inefficient, the quantitation of patient viral loads will be inaccurate. The laboratory administrator monitors the <strong>Slope</strong> to determine efficiency</p>
<ul>
<li>
<strong>The Relationship Between Slope and Efficiency</strong>
<ul>
<li>Ideally, a 10-fold dilution series (Log 10) should result in a <span class="math inline">\(C_t\)</span> difference of approximately <strong>3.32 cycles:</strong> between standards. (Because <span class="math inline">\(2^{3.32} \approx 10\)</span>)</li>
<li>Therefore, the <strong>Ideal Slope is -3.32</strong>
</li>
</ul>
</li>
<li>
<strong>Calculating Efficiency from Slope</strong>
<ul>
<li>The formula used by the software is: <span class="math inline">\(\text{Efficiency} (\%) = (10^{(-1/\text{slope})} - 1) \times 100\)</span>
</li>
<li>
<strong>Acceptance Criteria:</strong> Most clinical laboratories accept an efficiency range of <strong>90% to 110%</strong> (Slopes between -3.1 and -3.6)</li>
</ul>
</li>
<li>
<strong>Troubleshooting Slope Failures</strong>
<ul>
<li>
<strong>Low Efficiency (Slope &lt; -3.6, e.g., -3.9):</strong> The curve is “flat.” This indicates <strong>Inhibition</strong> (salt carryover, ethanol) or degrading reagents. The reaction is not doubling every cycle</li>
<li>
<strong>High Efficiency (Slope &gt; -3.1, e.g., -2.9):</strong> The curve is “steep.” This is often a mathematical artifact caused by pipetting errors in the serial dilution or primer-dimers artificially boosting the signal at low concentrations</li>
</ul>
</li>
</ul>
</div>
<div id="linearity-r2-precision" class="section level2 unnumbered">
<h2>Linearity (<span class="math inline">\(R^2\)</span>) &amp; Precision<a class="anchor" aria-label="anchor" href="#linearity-r2-precision"><i class="fas fa-link"></i></a>
</h2>
<p>While efficiency measures how well the reaction works, Linearity measures how consistent the pipetting and the dilution series were. It is the “Goodness of Fit.”</p>
<ul>
<li>
<strong>Coefficient of Determination (<span class="math inline">\(R^2\)</span>)</strong>
<ul>
<li>This statistical value indicates how closely the actual data points of the standards lie on the regression line</li>
<li>
<strong>Calculation:</strong> Computed automatically by the regression software. Values range from 0 to 1</li>
<li>
<strong>Acceptance Criteria:</strong> For clinical viral load assays, an <span class="math inline">\(R^2\)</span> value of <strong>&gt; 0.98</strong> or <strong>&gt; 0.99</strong> is typically required</li>
<li>
<strong>Failure Mode:</strong> If <span class="math inline">\(R^2\)</span> is 0.95, it implies significant pipetting error (scatter) occurred during the creation of the standards. The run must be rejected because the “ruler” is crooked</li>
</ul>
</li>
<li>
<strong>Precision of Replicates</strong>
<ul>
<li>Standards are often run in duplicate or triplicate. The software calculates the Standard Deviation of the <span class="math inline">\(C_t\)</span> values for each concentration level</li>
<li>
<strong>Calculation:</strong> If the replicates differ by more than <strong>0.5 <span class="math inline">\(C_t\)</span></strong>, the point is often excluded (outlier removal) to restore linearity, provided the SOP allows for this data manipulation</li>
</ul>
</li>
</ul>
</div>
<div id="the-lower-limit-of-quantification-lloq" class="section level2 unnumbered">
<h2>The Lower Limit of Quantification (LLoQ)<a class="anchor" aria-label="anchor" href="#the-lower-limit-of-quantification-lloq"><i class="fas fa-link"></i></a>
</h2>
<p>In Quality Management, calculating the “limit” is essential for reporting. There is a mathematical distinction between “detecting” a pathogen and “quantifying” it</p>
<ul>
<li>
<strong>Defining LLoQ</strong>
<ul>
<li>The LLoQ is the lowest concentration on the standard curve that maintains Linearity and Precision</li>
<li>
<strong>Calculation:</strong> It is the lowest standard where the Coefficient of Variation (%CV) of the calculated concentration is acceptable (typically &lt; 30% or &lt; 0.25 log)</li>
</ul>
</li>
<li>
<strong>Reporting Implications</strong>
<ul>
<li>If a patient sample has a <span class="math inline">\(C_t\)</span> value that falls <em>outside</em> the lowest point of the standard curve (extrapolation), the result cannot be reported numerically</li>
<li>
<strong>Result:</strong> “&lt; 20 copies/mL (Detected)” rather than “12 copies/mL.” This is because the linearity of the math model has not been proven below the lowest standard</li>
</ul>
</li>
</ul>
</div>
<div id="relative-quantification-deltadelta-c_t-method" class="section level2 unnumbered">
<h2>Relative Quantification (<span class="math inline">\(\Delta\Delta C_t\)</span> Method)<a class="anchor" aria-label="anchor" href="#relative-quantification-deltadelta-c_t-method"><i class="fas fa-link"></i></a>
</h2>
<p>While viral loads use Absolute Quantification (Standard Curves), gene expression studies (e.g., measuring mRNA levels of a tumor marker) often use <strong>Relative Quantification</strong>. This calculates the “Fold Change” of a target gene compared to a baseline, rather than an exact copy number</p>
<ul>
<li>
<strong>The <span class="math inline">\(\Delta C_t\)</span> Calculation (Normalization)</strong>
<ul>
<li>First, the target gene <span class="math inline">\(C_t\)</span> is normalized to a “Housekeeping Gene” (Reference) that does not change (e.g., GAPDH or Actin)</li>
<li><span class="math inline">\(\text{Equation:} \Delta C_t = C_t(\text{Target}) - C_t(\text{Reference})\)</span></li>
</ul>
</li>
<li>
<strong>The <span class="math inline">\(\Delta\Delta C_t\)</span> Calculation (Calibration)</strong>
<ul>
<li>The normalized value of the patient sample is compared to a “Calibrator” (e.g., a normal tissue sample or Time Zero sample)</li>
<li><span class="math inline">\(\text{Equation:} \Delta\Delta C_t = \Delta C_t(\text{Sample}) - \Delta C_t(\text{Calibrator})\)</span></li>
</ul>
</li>
<li>
<strong>The Fold Change</strong>
<ul>
<li>The final calculation determines the relative expression level</li>
<li><span class="math inline">\(\text{Equation:} \text{Fold Change} = 2^{-\Delta\Delta C_t}\)</span></li>
<li>
<strong>Interpretation:</strong> A result of “4” means the gene is expressed 4 times higher in the patient than in the normal control. A result of “0.5” means expression is downregulated by half</li>
</ul>
</li>
</ul>
</div>
<div id="digital-pcr-poisson-quantification" class="section level2 unnumbered">
<h2>Digital PCR (Poisson Quantification)<a class="anchor" aria-label="anchor" href="#digital-pcr-poisson-quantification"><i class="fas fa-link"></i></a>
</h2>
<p>A modern alternative to standard curves is Digital PCR (dPCR), which uses a different mathematical model for quantification. This is considered an advanced quality management calculation because it eliminates the need for a standard curve entirely</p>
<ul>
<li>
<strong>Partitioning</strong>
<ul>
<li>The sample is divided into 20,000+ microscopic droplets. Some droplets contain the DNA target (Positive), and some do not (Negative)</li>
</ul>
</li>
<li>
<strong>Poisson Statistics</strong>
<ul>
<li>Instead of <span class="math inline">\(C_t\)</span> values, the calculation is based on the <em>proportion</em> of negative partitions</li>
<li><span class="math inline">\(\text{Equation:} \lambda = -\ln(1 - p)\)</span></li>
<li>Where <span class="math inline">\(\lambda\)</span> is the average number of copies per droplet and <span class="math inline">\(p\)</span> is the fraction of positive droplets</li>
<li>
<strong>QM Benefit:</strong> This provides absolute quantification without the bias introduced by amplification efficiency or standard curve pipetting errors</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- Scoll to TOC element -->
<script defer>
  // adjust TOC scroll to selected element/topic
  function scroll_to_element() {
    header_height = document.getElementsByTagName('header')[0].offsetHeight/2;
    toc_element_position = document.querySelector('.book-toc > li > a.active').offsetTop;
    if(document.querySelector('.book-toc > li > a.active').offsetLeft == 0) {
      toc_element_position1 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetTop;
      toc_element_position2 = document.querySelector('.book-toc > li > a.active').offsetParent.offsetHeight;
      toc_element_position = toc_element_position1 + toc_element_position2;
    }
    document.getElementsByTagName('header')[0].scrollTo(0, toc_element_position - header_height, 'smooth');
  }
  window.onload = scroll_to_element();

  // hide those labes that take up multiple lines
  sections = [];
  topics = [];
  function hide_label() {
    // sections
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          elem.classList
          .contains('book-part')
          ? sections.push(elem)
          : '';
      });

    // topics
    document
      .querySelectorAll('.book-toc > li')
      .forEach(elem => {
        window
          .getComputedStyle(elem)
          .textTransform === "uppercase"
          &&
          !elem.classList
          .contains('book-part')
          ? topics.push(elem)
          : '';
      });
  }
  hide_label();
  head = document.getElementsByTagName('header')[0];
  head.addEventListener('scroll', () => {
    // sections
    count_sections = 0;
    prev_sections_elem = '';
    sections
      .forEach((elem) => {
      if(count_sections === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_sections = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_sections_elem.classList.add('hide_label');
        } else {
          prev_sections_elem.classList.remove('hide_label');
        }
      }
      prev_sections_elem = elem;
    });
    // topics
    count_topics = 0;
    prev_topic_elem = '';
    topics
      .forEach((elem) => {
      if(count_topics === 0) {
        initial_y = elem.getBoundingClientRect().y + 20;
        count_topics = 1;
      } else {
        if(elem.getBoundingClientRect().y < initial_y) {
          prev_topic_elem.classList.add('hide_label');
        } else {
          prev_topic_elem.classList.remove('hide_label');
        }
      }
      prev_topic_elem = elem;
    });
  });
</script><div class="chapter-nav">
<div class="prev"><a href="assay-validation-verification.html">Assay Validation &amp; Verification</a></div>
<div class="empty"></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#quantitative-calculations-1">Quantitative Calculations</a></li>
<li><a class="nav-link" href="#the-standard-curve-absolute-quantification">The Standard Curve (Absolute Quantification)</a></li>
<li><a class="nav-link" href="#pcr-efficiency-calculations">PCR Efficiency Calculations</a></li>
<li><a class="nav-link" href="#linearity-r2-precision">Linearity (\(R^2\)) &amp; Precision</a></li>
<li><a class="nav-link" href="#the-lower-limit-of-quantification-lloq">The Lower Limit of Quantification (LLoQ)</a></li>
<li><a class="nav-link" href="#relative-quantification-deltadelta-c_t-method">Relative Quantification (\(\Delta\Delta C_t\) Method)</a></li>
<li><a class="nav-link" href="#digital-pcr-poisson-quantification">Digital PCR (Poisson Quantification)</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/blob/main/Rmd/04-F-04-d.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettmrice/Clinical-Molecular-Biology/blob/edit/main/Rmd/04-F-04-d.Rmd" target="_blank">Edit this page <i class="fab fa-github"></i></a></li>
          <li><a id="buymeacoffee" href="https://buymeacoffee.com/bloodcellid" target="_blank">NEED Caffeine</a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
